<!doctype html><html lang=en><head><meta name=generator content="Hugo 0.148.2"><title>Carolina's CV</title><meta name=description content="Portfolio and personal CV of Carolina Monzó."><meta charset=UTF-8><meta name=viewport content="width=device-width,initial-scale=1"><meta http-equiv=X-UA-Compatible content="ie=edge"><link rel=stylesheet href=/application.228e9114d0768fdf2ebe316bcc9c20f3a463a83871918d61b1482f8de98636dd.css integrity="sha256-Io6RFNB2j98uvjFrzJwg86RjqDhxkY1hsUgvjemGNt0="><link rel=icon type=image/png href=/images/site/main-logo_hu_4e49bea04ab3bcc0.png><link rel=alternate type=application/rss+xml href=https://carolinamonzo.github.io/index.xml title="Carolina's CV"><meta property="og:title" content="Carolina's CV"><meta property="og:type" content="website"><meta property="og:description" content="Portfolio and personal CV of Carolina Monzó."><meta property="og:image" content="/images/author/CarolMonzoPictureProfile_small.png"><meta property="og:url" content="https://carolinamonzo.github.io"><script integrity="sha256-DO4ugzEwhTW1Id1UIWn0gUJWaebCYOypeTit6LW4QB4=">let theme=localStorage.getItem("theme-scheme")||localStorage.getItem("darkmode:color-scheme")||"light";theme==="system"&&(window.matchMedia&&window.matchMedia("(prefers-color-scheme: dark)").matches?theme="dark":theme="light"),document.documentElement.setAttribute("data-theme",theme)</script></head><body data-bs-spy=scroll data-bs-target=#top-navbar data-bs-offset=100><nav class="navbar navbar-expand-xl top-navbar shadow transparent-navbar homepage" id=top-navbar><div class=container><a class=navbar-brand href=/><img src=/images/site/inverted-logo_hu_ee96727f9c98e7c5.png id=logo alt=Logo>
Carolina's CV</a>
<button class="navbar-toggler navbar-dark" id=navbar-toggler type=button data-bs-toggle=collapse data-bs-target=#top-nav-items aria-label=menu>
<i data-feather=menu></i></button><div class="collapse navbar-collapse dynamic-navbar" id=top-nav-items><ul class="nav navbar-nav ms-auto"><li class=nav-item><a class=nav-link href=#home>Home</a></li><li class=nav-item><a class=nav-link href=#about>About</a></li><li class=nav-item><a class=nav-link href=#skills>Skills</a></li><li class=nav-item><a class=nav-link href=#experiences>Experience</a></li><li class=nav-item><a class=nav-link href=#education>Education</a></li><li class=nav-item><a class=nav-link href=#projects>Grants and projects</a></li><li class=nav-item><a class=nav-link href=#achievements>Awards</a></li><li class=nav-item><a class=nav-link href=#publications>Publications</a></li><li class=nav-item><a class=nav-link href=#accomplishments>Courses</a></li><li class="nav-item dropdown"><a class="nav-link dropdown-toggle" href=# id=navbarDropdown role=button data-bs-toggle=dropdown aria-haspopup=true aria-expanded=false>More</a><div class=dropdown-menu aria-labelledby=navbarDropdown><a class=dropdown-item href=#outreach>Outreach</a>
<a class=dropdown-item href=#featured-posts>Other</a></div></li><div id=top-navbar-divider></div><li class="nav-item dropdown"><a class="nav-link dropdown-toggle" href=# id=themeSelector role=button data-bs-toggle=dropdown aria-haspopup=true aria-expanded=false><img id=navbar-theme-icon-svg class=theme-icon src=/icons/moon-svgrepo-com.svg width=20 alt="Dark Theme"></a><div id=themeMenu class="dropdown-menu dropdown-menu-icons-only" aria-labelledby=themeSelector><a class="dropdown-item nav-link" href=# data-scheme=light><img class=theme-icon src=/icons/sun-svgrepo-com.svg width=20 alt="Light Theme">
</a><a class="dropdown-item nav-link" href=# data-scheme=dark><img class=theme-icon src=/icons/moon-svgrepo-com.svg width=20 alt="Dark Theme">
</a><a class="dropdown-item nav-link" href=# data-scheme=system><img class=theme-icon src=/icons/computer-svgrepo-com.svg width=20 alt="System Theme"></a></div></li></ul></div></div><img src=/images/site/main-logo_hu_4e49bea04ab3bcc0.png class=d-none id=main-logo alt=Logo>
<img src=/images/site/inverted-logo_hu_ee96727f9c98e7c5.png class=d-none id=inverted-logo alt="Inverted Logo"></nav><div class="container-fluid home" id=home><style>#homePageBackgroundImageDivStyled{background-image:url(/images/site/blackandwhiteValencia_hu_a40e5d68b56d18e3.png)}@media(min-width:500px) and (max-width:800px){#homePageBackgroundImageDivStyled{background-image:url(/images/site/blackandwhiteValencia_hu_bd379e8e5801dee3.png)}}@media(min-width:801px) and (max-width:1200px){#homePageBackgroundImageDivStyled{background-image:url(/images/site/blackandwhiteValencia_hu_ce86a27f91fb71dc.png)}}@media(min-width:1201px) and (max-width:1500px){#homePageBackgroundImageDivStyled{background-image:url(/images/site/blackandwhiteValencia_hu_466c479d3ced5a5f.png)}}@media(min-width:1501px){#homePageBackgroundImageDivStyled{background-image:url(/images/site/blackandwhiteValencia.png)}}html[data-theme=dark]{ #homePageBackgroundImageDivStyled { background-image: url('/images/site/blackandwhiteValencia_hu_a40e5d68b56d18e3.png'); } @media (min-width: 500px) and (max-width: 800px) { #homePageBackgroundImageDivStyled { background-image: url('/images/site/blackandwhiteValencia_hu_bd379e8e5801dee3.png'); } } @media (min-width: 801px) and (max-width: 1200px) { #homePageBackgroundImageDivStyled { background-image: url('/images/site/blackandwhiteValencia_hu_ce86a27f91fb71dc.png'); } } @media (min-width: 1201px) and (max-width: 1500px) { #homePageBackgroundImageDivStyled { background-image: url('/images/site/blackandwhiteValencia_hu_466c479d3ced5a5f.png'); } } @media (min-width: 1501px) { #homePageBackgroundImageDivStyled { background-image: url('/images/site/blackandwhiteValencia.png'); } }}</style><span class=on-the-fly-behavior></span><div id=homePageBackgroundImageDivStyled class="background container-fluid"></div><div class="container content text-center"><img src=/images/author/CarolMonzoPictureProfile_small_hu_a267923b85887895.png class="rounded-circle mx-auto d-block img-fluid" alt="Author Image"><h1 class=greeting>Hi, I am Carolina</h1><div class=typing-carousel><span id=ityped class=ityped></span>
<span class=ityped-cursor></span></div><ul id=typing-carousel-data><li>I am a biologist</li><li>I am a data analyst</li><li>I love statistics</li><li>I work on developing tools for the analysis of genomic and transcriptomic data</li><li>I love to work using statistics to answer biological questions</li></ul><a href=#about class=arrow-center aria-label="Read More - Carolina"><i class="arrow bounce fa fa-chevron-down"></i></a></div></div><div class="container-fluid section-holder d-flex bg-primary"><div class="container anchor p-lg-5 about-section" id=about><div class="row pt-sm-2 pt-md-4 align-self-center"><div class=col-sm-6><h3 class=p-1>Carolina Monzó</h3><h5 class=p-1>MSCA Postdoctoral Fellow
at <a href=https://i2sysbio.es/ title="Institute for Integrative Systems Biology" target=_blank rel=noopener>Institute for Integrative Systems Biology</a></h5><p class="p-1 text-justify">I am a passionate bioinformatician and data analyst with 9 years of working experience. I analyzed cross-sectional and longitudinal phenotypic and omic data. I also built software tools for bioinformaticians using Python and R. Sometimes, I performed wet-lab experiments.</p><div class="text-container ms-auto"><ul class="social-link d-flex"><li><a href=mailto:carolina.monzo.age@gmail.com title=Email target=_blank rel=noopener><i class="fas fa-envelope"></i></a></li><li><a href=https://www.github.com/carolinamonzo title=Github target=_blank rel=noopener><i class="fab fa-github"></i></a></li><li><a href=https://www.linkedin.com/in/carolina-monz%C3%B3-944762134/ title=LinkedIn target=_blank rel=noopener><i class="fab fa-linkedin"></i></a></li><li><a href="https://www.researchgate.net/profile/Carolina-Monzo?ev=hdr_xprf" title=ResearchGate target=_blank rel=noopener><i class="fab fa-researchgate"></i></a></li></ul></div><a href=/files/resume.pdf title="My Resume" target=#><button class="btn btn-dark">My Resume</button></a></div><div class="col-sm-6 pt-5 ps-md-4 ps-sm-3 pt-sm-0"><div class=row><div class="col-6 col-lg-4 p-2"><div class="circular-progress blue"><span class=circular-progress-left><span class="circular-progress-bar circular-progress-percentage-35 circular-progress-percentage-50-delay"></span>
</span><span class=circular-progress-right><span class="circular-progress-bar circular-progress-percentage-50"></span></span><div class=circular-progress-value>Statistics</div></div></div><div class="col-6 col-lg-4 p-2"><div class="circular-progress yellow"><span class=circular-progress-left><span class="circular-progress-bar circular-progress-percentage-40 circular-progress-percentage-50-delay"></span>
</span><span class=circular-progress-right><span class="circular-progress-bar circular-progress-percentage-50"></span></span><div class=circular-progress-value>Aging</div></div></div><div class="col-6 col-lg-4 p-2"><div class="circular-progress orange"><span class=circular-progress-left><span class="circular-progress-bar circular-progress-percentage-35 circular-progress-percentage-50-delay"></span>
</span><span class=circular-progress-right><span class="circular-progress-bar circular-progress-percentage-50"></span></span><div class=circular-progress-value>Immune system</div></div></div></div></div></div></div></div><div class="container-fluid section-holder d-flex bg-secondary"><div class="container-fluid anchor pb-5 skills-section" id=skills><h1 class=text-center><span id=skills></span>Skills</h1><div class="container ms-auto text-center"><div class="btn-group flex-wrap" role=group id=skill-filter-buttons><button type=button class="btn btn-dark skill-filtr-control" data-filter=all>
All
</button>
<button type=button class="btn btn-dark skill-filtr-control" data-filter=basic>
Basic
</button>
<button type=button class="btn btn-dark skill-filtr-control" data-filter=language>
Language
</button>
<button type=button class="btn btn-dark skill-filtr-control" data-filter=wet-lab>
Wet-lab
</button>
<button type=button class="btn btn-dark skill-filtr-control" data-filter=others>
Others</button></div></div><div class="container d-flex-block filtr-skills"><div class=row id=skill-card-holder style=margin-left:unset><div class="col-xs-12 col-sm-6 col-lg-4 pt-2 filtr-item" data-category='all, basic'><a class=text-decoration-none href=https://git-scm.com/ title=Git target=_blank rel=noopener><div class=card><div class="card-head d-flex"><img class=card-img-xs src=/images/sections/skills/git_hu_15d77950cb79075d.png alt=Git><h5 class=card-title>Git</h5></div><div class="card-body text-justify pt-1 pb-1"><p class=card-text>Experienced with git-based development. Mostly, use Github. Also, have experience in working with GitLab.</p></div></div></a></div><div class="col-xs-12 col-sm-6 col-lg-4 pt-2 filtr-item" data-category='all, basic'><a class=text-decoration-none><div class=card><div class="card-head d-flex"><img class=card-img-xs src=/images/sections/skills/linux_hu_1dbc6391c584bd2f.png alt=Linux><h5 class=card-title>Linux</h5></div><div class="card-body text-justify pt-1 pb-1"><p class=card-text>Using as the main operating system. Capable of writing bash/shell scripts.</p></div></div></a></div><div class="col-xs-12 col-sm-6 col-lg-4 pt-2 filtr-item" data-category='all, basic,language'><a class=text-decoration-none href=https://www.python.org/ title="Python Development" target=_blank rel=noopener><div class=card><div class="card-head d-flex"><img class=card-img-xs src=/images/sections/skills/python.svg alt="Python Development"><h5 class=card-title>Python Development</h5></div><div class="card-body text-justify pt-1 pb-1"><p class=card-text>Using as the main language for professional development. Capable of writing scalable, testable, and maintainable programs.</p></div></div></a></div><div class="col-xs-12 col-sm-6 col-lg-4 pt-2 filtr-item" data-category='all, basic,language'><a class=text-decoration-none href=https://www.r-project.org/ title="R Development" target=_blank rel=noopener><div class=card><div class="card-head d-flex"><img class=card-img-xs src=/images/sections/skills/R_hu_3477d6c301796bfa.png alt="R Development"><h5 class=card-title>R Development</h5></div><div class="card-body text-justify pt-1 pb-1"><p class=card-text>Contributed a package to Bioconductor. Capable of writing testable, and maintainable programs.</p></div></div></a></div><div class="col-xs-12 col-sm-6 col-lg-4 pt-2 filtr-item" data-category='all, others'><a class=text-decoration-none><div class=card><div class="card-head d-flex"><img class=card-img-xs src=/images/sections/skills/cloud_hu_3f5df1f90a46cbf5.png alt="Cloud Computing"><h5 class=card-title>Cloud Computing</h5></div><div class="card-body text-justify pt-1 pb-1"><p class=card-text>Basic knowledge of AWS infrastructure.</p></div></div></a></div><div class="col-xs-12 col-sm-6 col-lg-4 pt-2 filtr-item" data-category='all, others'><a class=text-decoration-none><div class=card><div class="card-head d-flex"><img class=card-img-xs src=/images/sections/skills/mouse_hu_e91e5aecc1e90ae.png alt="Experiment Design"><h5 class=card-title>Experiment Design</h5></div><div class="card-body text-justify pt-1 pb-1"><p class=card-text>Experienced designing mouse and tissue-based experiments to answer biological questions or generate ground truth datasets.</p></div></div></a></div><div class="col-xs-12 col-sm-6 col-lg-4 pt-2 filtr-item" data-category='all, wet-lab'><a class=text-decoration-none><div class=card><div class="card-head d-flex"><img class=card-img-xs src=/images/sections/skills/metabolomics_hu_ba66d4b2181d4ea9.png alt="Nucleic Acid extraction"><h5 class=card-title>Nucleic Acid extraction</h5></div><div class="card-body text-justify pt-1 pb-1"><p class=card-text>Experienced extracting DNA, RNA, proteins and metabolites from tissue samples.</p></div></div></a></div><div class="col-xs-12 col-sm-6 col-lg-4 pt-2 filtr-item" data-category='all, wet-lab'><a class=text-decoration-none><div class=card><div class="card-head d-flex"><img class=card-img-xs src=/images/sections/skills/eppendorf_hu_d09e4468290172d8.png alt="Library Preparation"><h5 class=card-title>Library Preparation</h5></div><div class="card-body text-justify pt-1 pb-1"><p class=card-text>Experienced designing and optimizing experiments for sequencing using Sanger, Illumina, PacBio and ONT sequencing technologies.</p></div></div></a></div><div class="col-xs-12 col-sm-6 col-lg-4 pt-2 filtr-item" data-category='all, others'><a class=text-decoration-none><div class=card><div class="card-head d-flex"><img class=card-img-xs src=/images/sections/skills/omics_hu_66380ad11005d25b.png alt="Omic Data Analysis"><h5 class=card-title>Omic Data Analysis</h5></div><div class="card-body text-justify pt-1 pb-1"><p class=card-text>Experienced analyzing genomic, transcriptomic, proteomic, metabolomic, metataxonomic and metagenomic datasets.</p></div></div></a></div></div></div></div></div><div class="container-fluid section-holder d-flex bg-primary"><div class="container-fluid anchor pb-5 experiences-section"><h1 class=text-center><span id=experiences></span>Experience</h1><div class="container timeline text-justify"><div class="row align-items-center d-flex"><div class="col-1 col-lg-2 text-center vertical-line d-inline-flex justify-content-center"><div class="circle fw-bold">1</div></div><div class="col-10 col-lg-8"><div class=experience-entry-heading><div class=logo-holder><img class="company-logo light-logo" src=/images/sections/experiences/company1.png alt></div><div class=company-heading><h5><a href=https://i2sysbio.es/ title="Institute for Integrative Systems Biology (I2SysBio)" target=_blank rel=noopener>Institute for Integrative Systems Biology (I2SysBio)</a></h5><p class=text-muted>Sep 2022 - Present</p></div><p class=text-muted><i class="fa-solid fa-location-dot"></i> Valencia, Spain</p><p>The I2SysBio is a research institute of the Spanish National Research Institute (CSIC) focusing on basic and systems biology.</p></div><div class=positions><div class=company-heading><h5 class=designation>Marie Skłodowska-Curie Actions Postdoctoral Fellow</h5><p class=text-muted>Sep 2024 - Present</p></div><h6 class=text-heading>Responsibilities:</h6><ul class=justify-content-around><li>Design and develop tools for long-reads transcriptomics data analysis.</li><li>Supervise PhD and MSc students.</li><li>Design studies and apply for independent funding.</li><li>Represent postdocs in the institute board.</li><li>Member of the training committee.</li><li>Co-lead the Junior Hub on Bioinformatics and Computational Biology of CSIC.</li></ul><div class=company-heading><h5 class=designation>Postdoctoral Researcher</h5><p class=text-muted>Sep 2022 - Sep 2024</p></div><h6 class=text-heading>Responsibilities:</h6><ul class=justify-content-around><li>Design and develop tools for single-cell multi-omics data simulation and analysis.</li><li>Supervise MSc students, organize seminars, manage social media.</li><li>Design studies and collaborate in funding applications.</li></ul></div></div></div><div class="row horizontal-line"><div class="col-1 col-lg-2 timeline-side-div"><div class=corner></div></div><div class="col-10 col-lg-8"><hr></div><div class="col-1 col-lg-2 timeline-side-div"><div class=corner></div></div></div><div class="row align-items-center justify-content-end d-flex"><div class="col-10 col-lg-8"><div class=experience-entry-heading><div class=logo-holder><img class="company-logo light-logo" src=/images/sections/experiences/company2.png alt></div><div class=company-heading><h5><a href=https://www.age.mpg.de/en title="Max Planck Institute for Biology of Aging (MPI-AGE)." target=_blank rel=noopener>Max Planck Institute for Biology of Aging (MPI-AGE).</a></h5><p class=text-muted>Feb 2019 - Nov 2022</p></div><p class=text-muted><i class="fa-solid fa-location-dot"></i> Cologne, Germany</p><p>The MPI-AGE is one of the world leading research institutes on aging focusing on understanding the natural aging process and prevent age-related diseases.</p></div><div class=positions><div class=company-heading><h5 class=designation>Postdoctoral Researcher</h5><p class=text-muted>Oct 2022 - Nov 2022</p></div><h6 class=text-heading>Responsibilities:</h6><ul class=justify-content-around><li>Write scientific articles.</li></ul><div class=company-heading><h5 class=designation>PhD Student</h5><p class=text-muted>Feb 2019 - Oct 2022</p></div><h6 class=text-heading>Responsibilities:</h6><ul class=justify-content-around><li>Multiple omics data analysis, cross-sectional and longitudinal statistical analysis.</li><li>DNA, RNA, proteins and metabolites extraction. Library preparation and sequencing.</li><li>Study aging and anti-aging interventions in mice, focusing on the microbiome and adaptive immune system.</li></ul></div></div><div class="col-1 col-lg-2 text-center vertical-line d-inline-flex justify-content-center"><div class="circle fw-bold">2</div></div></div><div class="row horizontal-line"><div class="col-1 col-lg-2 timeline-side-div"><div class=corner></div></div><div class="col-10 col-lg-8"><hr></div><div class="col-1 col-lg-2 timeline-side-div"><div class=corner></div></div></div><div class="row align-items-center d-flex"><div class="col-1 col-lg-2 text-center vertical-line d-inline-flex justify-content-center"><div class="circle fw-bold">3</div></div><div class="col-10 col-lg-8"><div class=experience-entry-heading><div class=logo-holder><img class="company-logo light-logo" src=/images/sections/experiences/company3.png alt></div><div class=company-heading><h5><a href=https://www.incliva.es/en/ title="Health Research Institute (INCLIVA)." target=_blank rel=noopener>Health Research Institute (INCLIVA).</a></h5><p class=text-muted>Jun 2017 - Dec 2018</p></div><p class=text-muted><i class="fa-solid fa-location-dot"></i> Valencia, Spain</p><p>Research foundation of the Clinical University Hospital, performs translational research and clinical diagnosis.</p></div><div class=positions><div class=company-heading><h5 class=designation>Senior Technician</h5><p class=text-muted>Jun 2017 - Dec 2018</p></div><h6 class=text-heading>Responsibilities:</h6><ul class=justify-content-around><li>DNA extraction, amplicon and capture probe library preparation for Sanger and Illumina Sequencing.</li><li>DNA variation analysis and interpretation.</li><li>Population genetic studies.</li><li>Bioinformatic pipeline development for amplicon, exome and Copy Number Variants analysis.</li></ul></div></div></div><div class="row horizontal-line"><div class="col-1 col-lg-2 timeline-side-div"><div class=corner></div></div><div class="col-10 col-lg-8"><hr></div><div class="col-1 col-lg-2 timeline-side-div"><div class=corner></div></div></div><div class="row align-items-center justify-content-end d-flex"><div class="col-10 col-lg-8"><div class=experience-entry-heading><div class=logo-holder><img class="company-logo light-logo" src=/images/sections/experiences/company4.png alt></div><div class=company-heading><h5><a href=https://healthincode.com/en/ title="Health in Code (Previously IMEGEN)." target=_blank rel=noopener>Health in Code (Previously IMEGEN).</a></h5><p class=text-muted>Jan 2017 - Jun 2017</p></div><p class=text-muted><i class="fa-solid fa-location-dot"></i> Valencia, Spain</p><p>Spanish leading company in the field of medical genetics.</p></div><div class=positions><div class=company-heading><h5 class=designation>Bioinformatics Technician</h5><p class=text-muted>Jan 2017 - Jun 2017</p></div><h6 class=text-heading>Responsibilities:</h6><ul class=justify-content-around><li>Bioinformatics pipeline development and maintenance.</li><li>Variant prioritization.</li><li>Provide bioinformatic analysis support to the R&amp;D department.</li></ul></div></div><div class="col-1 col-lg-2 text-center vertical-line d-inline-flex justify-content-center"><div class="circle fw-bold">4</div></div></div></div></div></div><div class="container-fluid section-holder d-flex bg-secondary"><div class="container-fluid anchor pb-5 education-section"><h1 class=text-center><span id=education></span>Education</h1><div class=container><table class=education-info-table><tbody><tr><td class=icon><div class=hline></div><div class=icon-holder><i class="fas fa-graduation-cap"></i></div></td><td class=line><div></div></td><td class=details><div class="degree-info card"><div class=logo-holder><img class="company-logo light-logo" src=/images/sections/education/tech_college.png alt="Ph.D in Natural Sciences"></div><div class=row><div class="col-lg-10 col-md-8"><h5><a href=https://www.uni-koeln.de/ title="Universität zu Köln" target=_blank rel=noopener>Universität zu Köln</a></h5></div><div class="timeframe col-lg-2 col-md-4">01/02/2019-21/10/2022</div></div><h6>Ph.D in Natural Sciences</h6><div class=custom-section><h6 class=text-heading>Thesis title:</h6><div class=custom-content>The effect of Dietary Restriction on the Microbiome and the Adaptive Immune System in Mouse Ageing</div><h6 class=text-heading>Grade:</h6><div class=custom-content>Outstanding</div><h6 class=text-heading>Supervisors:</h6><div class=custom-content>Prof. Dame Linda Partridge and Prof. Andreas Beyer</div><h6 class=text-heading>url:</h6><div class=custom-content><a href=https://kups.ub.uni-koeln.de/70176/ target=_blank rel=noopener>https://kups.ub.uni-koeln.de/70176/</a></div><h6 class=text-heading>Location:</h6><div class=custom-content>Max Planck Institute for Biology of Ageing, Department of Biological Mechanisms of Aging</div></div></div></td></tr><tr><td class=icon><div class=hline></div><div class=icon-holder><i class="fas fa-graduation-cap"></i></div></td><td class=line><div></div></td><td class=details><div class="degree-info card"><div class=logo-holder><img class="company-logo light-logo" src=/images/sections/education/university.png alt="M.Sc. in Bioinformatics"></div><div class=row><div class="col-lg-10 col-md-8"><h5><a href=https://www.uv.es/ title="Universitat de València" target=_blank rel=noopener>Universitat de València</a></h5></div><div class="timeframe col-lg-2 col-md-4">01/09/2016-01/10/2018</div></div><h6>M.Sc. in Bioinformatics</h6><div class=custom-section><h6 class=text-heading>Thesis title:</h6><div class=custom-content>Study of Genetic Alterations in Patients with Imprinting Diseases</div><h6 class=text-heading>Grade:</h6><div class=custom-content>Outstanding</div><h6 class=text-heading>Supervisors:</h6><div class=custom-content>Dr. Javier Chaves and Dr. Guiomar Pérez de Nanclares</div><h6 class=text-heading>Location:</h6><div class=custom-content>INCLIVA, Genomics and Genetic Diagnosis Unit</div></div></div></td></tr><tr><td class=icon><div class=hline></div><div class=icon-holder><i class="fas fa-graduation-cap"></i></div></td><td class=line><div></div></td><td class=details><div class="degree-info card"><div class=logo-holder><img class="company-logo light-logo" src=/images/sections/education/university.png alt="Postgraduate Diploma in Medical Genetics"></div><div class=row><div class="col-lg-10 col-md-8"><h5><a href=https://www.uv.es/ title="Universitat de València" target=_blank rel=noopener>Universitat de València</a></h5></div><div class="timeframe col-lg-2 col-md-4">01/09/2015-01/07/2016</div></div><h6>Postgraduate Diploma in Medical Genetics</h6><div class=custom-section><h6 class=text-heading>Location:</h6><div class=custom-content>Príncipe Felipe Research Center, Computational Genomics Department</div></div></div></td></tr><tr><td class=icon><div class=hline></div><div class=icon-holder><i class="fas fa-graduation-cap"></i></div></td><td class=line><div></div></td><td class=details><div class="degree-info card"><div class=logo-holder><img class="company-logo light-logo" src=/images/sections/education/university.png alt="B.Sc. in Biology"></div><div class=row><div class="col-lg-10 col-md-8"><h5><a href=https://www.uv.es/ title="Universitat de València" target=_blank rel=noopener>Universitat de València</a></h5></div><div class="timeframe col-lg-2 col-md-4">01/09/2011-01/07/2016</div></div><h6>B.Sc. in Biology</h6><div class=custom-section><h6 class=text-heading>Thesis title:</h6><div class=custom-content>Characterization of CNVs in the Valencian Population, identified using Whole Genome Arrays</div><h6 class=text-heading>Grade:</h6><div class=custom-content>Excellent</div><h6 class=text-heading>Supervisor:</h6><div class=custom-content>Dr. Raquel Rodríguez-López</div><h6 class=text-heading>Location:</h6><div class=custom-content>General University Hospital of Valencia, Genetics Diagnosis Unit</div></div></div></td></tr></tbody></table></div></div></div><div class="container-fluid section-holder d-flex bg-primary"><div class="container-fluid anchor pb-5 projects-section" id=projects><h1 class=text-center><span id=projects></span>Grants and projects</h1><div class="container ms-auto text-center"><div class="btn-group flex-wrap" role=group id=project-filter-buttons><button type=button class="btn btn-dark project-filtr-control" data-filter=all>
All
</button>
<button type=button class="btn btn-dark project-filtr-control" data-filter="software development">
software development
</button>
<button type=button class="btn btn-dark project-filtr-control" data-filter=wet-lab>
wet-lab</button></div></div><div class="container filtr-projects"><div class=row id=project-card-holder><div class="col-sm-12 col-md-6 col-lg-4 p-2 filtr-item" data-category='all, wet-lab,software development,Oxford Nanopore Technologies'><div class="card mt-1"><div class=card><a href=https://nanoporetech.com/support/customer-support/my-account/what-is-the-oxford-nanopore-grant-programme- target=_blank rel=noopener><div class=card-header><div><div class=d-flex><img class=card-img-xs src=/images/sections/projects/ont_hu_4f43a3eb5835c656.png alt="Oxford Nanopore Grant"><h5 class="card-title mb-0">Oxford Nanopore Grant</h5></div><div class=sub-title><span>Principal Investigator</span>
<span>July 2016 - Present</span></div></div></div></a><div class="card-body text-justify pt-1 pb-1"><p>Grant for a Oxford Nanopore PromethION P2 Solo pack including sequencer and reagents. Investigating RNA degradation patterns using Nanopore long-read sequencing.</p><div class=project-card-footer><div class=project-tags-holder><span class="badge btn-info">wet-lab
</span><span class="badge btn-info">software development
</span><span class="badge btn-info">Oxford Nanopore Technologies</span></div><div class=project-btn-holder><span><a class="btn btn-outline-info btn-sm" href=https://nanoporetech.com/support/customer-support/my-account/what-is-the-oxford-nanopore-grant-programme- target=#>Details</a></span></div></div></div></div></div></div><div class="col-sm-12 col-md-6 col-lg-4 p-2 filtr-item" data-category='all, software development,machine-learning,CSIC'><div class="card mt-1"><div class=card><a href=https://sede.csic.gob.es/tramites/programa-jae/jae-intro-2025 target=_blank rel=noopener><div class=card-header><div><div class=d-flex><img class=card-img-xs src=/images/sections/projects/csic_hu_47f45042c86cfce.jpg alt="JAE Intro 2025"><h5 class="card-title mb-0">JAE Intro 2025</h5></div><div class=sub-title><span>Principal Investigator</span>
<span>Sep 2025 - Mar 2025</span></div></div></div></a><div class="card-body text-justify pt-1 pb-1"><p>Grant for developing software for differentiating between technical noise and transcript divergency in long-reads transcriptomics.</p><div class=project-card-footer><div class=project-tags-holder><span class="badge btn-info">software development
</span><span class="badge btn-info">machine-learning
</span><span class="badge btn-info">CSIC</span></div><div class=project-btn-holder><span><a class="btn btn-outline-info btn-sm" href=https://sede.csic.gob.es/tramites/programa-jae/jae-intro-2025 target=#>Details</a></span></div></div></div></div></div></div><div class="col-sm-12 col-md-6 col-lg-4 p-2 filtr-item" data-category='all, software development,European Comision'><div class="card mt-1"><div class=card><a href=https://github.com/ConesaLab/SQANTI3 target=_blank rel=noopener><div class=card-head><img class=card-img-top src=/images/sections/projects/sqantireads.svg alt="Marie Skłodowska-Curie Actions Postdoctoral Fellowship"></div><div class=card-header><div><div class=d-flex><img class=card-img-xs src=/images/sections/projects/marie_curie1_hu_b027bbdcc9704837.jpg alt="Marie Skłodowska-Curie Actions Postdoctoral Fellowship"><h5 class="card-title mb-0">Marie Skłodowska-Curie Actions Postdoctoral Fellowship</h5></div><div class=sub-title><span>Principal Investigator</span>
<span>Sep 2024 - Sep 2026</span></div></div></div></a><div class="card-body text-justify pt-1 pb-1"><p>Grant for deciphering the role of RNA biological noise in aging using long-reads transcriptomics.</p><div class=project-card-footer><div class=project-tags-holder><span class="badge btn-info">software development
</span><span class="badge btn-info">European Comision</span></div><div class=project-btn-holder><a class="github-button project-btn d-none" href=https://github.com/ConesaLab/SQANTI3 data-icon=octicon-standard data-show-count=true aria-label="Star Marie Skłodowska-Curie Actions Postdoctoral Fellowship">Star</a></div></div></div></div></div></div><div class="col-sm-12 col-md-6 col-lg-4 p-2 filtr-item" data-category='all, software development,Ministry of Science'><div class="card mt-1"><div class=card><a href=https://github.com/ConesaLab/MOSim target=_blank rel=noopener><div class=card-head><img class=card-img-top src=/images/sections/projects/bioconductor_hu_c99f52c5aa13ad2c.png alt="National Plan 2020"></div><div class=card-header><div><div class=d-flex><img class=card-img-xs src=/images/sections/projects/micin_hu_d987090461123118.jpg alt="National Plan 2020"><h5 class="card-title mb-0">National Plan 2020</h5></div><div class=sub-title><span>Contributor</span>
<span>Sep 2022 - Sep 2024</span></div></div></div></a><div class="card-body text-justify pt-1 pb-1"><p>Development of bulk and single-cell multi-omics simulation software.</p><div class=project-card-footer><div class=project-tags-holder><span class="badge btn-info">software development
</span><span class="badge btn-info">Ministry of Science</span></div><div class=project-btn-holder><a class="github-button project-btn d-none" href=https://github.com/ConesaLab/MOSim data-icon=octicon-standard data-show-count=true aria-label="Star National Plan 2020">Star</a></div></div></div></div></div></div><div class="col-sm-12 col-md-6 col-lg-4 p-2 filtr-item" data-category='all, wet-lab,software development,statistics'><div class="card mt-1"><div class=card><a href="https://github.com/carolinamonzo?tab=repositories" target=_blank rel=noopener><div class=card-header><div><div class=d-flex><img class=card-img-xs src=/images/sections/projects/mpi_hu_4971dbcc6ee9ee5f.jpg alt="Max Planck Society"><h5 class="card-title mb-0">Max Planck Society</h5></div><div class=sub-title><span>Contributor</span>
<span>Feb 2019 - Nov 2022</span></div></div></div></a><div class="card-body text-justify pt-1 pb-1"><p>Bioinformatic pipeline development and statistical analysis for cross-sectional and longitudinal omics data.</p><div class=project-card-footer><div class=project-tags-holder><span class="badge btn-info">wet-lab
</span><span class="badge btn-info">software development
</span><span class="badge btn-info">statistics</span></div><div class=project-btn-holder><a class="github-button project-btn d-none" href="https://github.com/carolinamonzo?tab=repositories" data-icon=octicon-standard data-show-count=true aria-label="Star Max Planck Society">Star</a></div></div></div></div></div></div><div class="col-sm-12 col-md-6 col-lg-4 p-2 filtr-item" data-category='all, software development,statistics,Regional funding'><div class="card mt-1"><div class=card><a href=javascript:void(0)><div class=card-header><div><div class=d-flex><img class=card-img-xs src=/images/sections/projects/valencia.svg alt="City of Valencia Scholarship"><h5 class="card-title mb-0">City of Valencia Scholarship</h5></div><div class=sub-title><span>Granted Fellow</span>
<span>Jan 2017 - Jun 2017</span></div></div></div></a><div class="card-body text-justify pt-1 pb-1"><p>Bioinformatic pipeline development for genomic variant prioritization.</p><div class=project-card-footer><div class=project-tags-holder><span class="badge btn-info">software development
</span><span class="badge btn-info">statistics
</span><span class="badge btn-info">Regional funding</span></div><div class=project-btn-holder></div></div></div></div></div></div><div class="col-sm-12 col-md-6 col-lg-4 p-2 filtr-item" data-category='all, Administration,Regional funding'><div class="card mt-1"><div class=card><a href=javascript:void(0)><div class=card-header><div><div class=d-flex><img class=card-img-xs src=/images/sections/projects/valencia.svg alt="La Dipu te Beca"><h5 class="card-title mb-0">La Dipu te Beca</h5></div><div class=sub-title><span>Granted Fellow</span>
<span>Jun 2014</span></div></div></div></a><div class="card-body text-justify pt-1 pb-1"><p>Accounting support to Petrés Town Hall.</p><div class=project-card-footer><div class=project-tags-holder><span class="badge btn-info">Administration
</span><span class="badge btn-info">Regional funding</span></div><div class=project-btn-holder></div></div></div></div></div></div><div class="col-sm-12 col-md-6 col-lg-4 p-2 filtr-item" data-category='all, Administration,Regional funding'><div class="card mt-1"><div class=card><a href=javascript:void(0)><div class=card-header><div><div class=d-flex><img class=card-img-xs src=/images/sections/projects/valencia.svg alt="La Dipu te Beca"><h5 class="card-title mb-0">La Dipu te Beca</h5></div><div class=sub-title><span>Granted Fellow</span>
<span>Jun 2012</span></div></div></div></a><div class="card-body text-justify pt-1 pb-1"><p>Accounting support to Petrés Town Hall.</p><div class=project-card-footer><div class=project-tags-holder><span class="badge btn-info">Administration
</span><span class="badge btn-info">Regional funding</span></div><div class=project-btn-holder></div></div></div></div></div></div></div></div></div></div><div class="container-fluid section-holder d-flex bg-secondary"><div class="container-fluid anchor pb-5 achievements-section"><h1 class=text-center><span id=achievements></span>Awards</h1><div class=container><div class=row id=gallery></div></div><div class=d-none id=achievements-holder><div class="achievement-entry text-center" style=background-image:url(/images/sections/achievements/keynote_hu_6d4589bc90509b73.jpeg)><i class="fa-solid fa-xmark hidden"></i>
<i class="fa-solid fa-magnifying-glass-plus" id=enlarge-icon></i><h4 class=title id=achievement-title>Keynote Speaker</h4><div class="caption hidden col-lg-6 text-start" id=caption><h4>Keynote Speaker</h4><p>IX Spanish Symposium of Bioinformatics Students.</p></div><span style=background-image:url(/images/sections/achievements/keynote_hu_6d4589bc90509b73.jpeg) class=d-none id=SmallImage active=true></span>
<span style=background-image:url(/images/sections/achievements/keynote_hu_9c16cf155465cb31.jpeg) class=d-none id=LargeImage></span></div><div class="achievement-entry text-center" style=background-image:url(/images/sections/achievements/flashtalk_hu_5709965b2f901028.jpeg)><i class="fa-solid fa-xmark hidden"></i>
<i class="fa-solid fa-magnifying-glass-plus" id=enlarge-icon></i><h4 class=title id=achievement-title>2nd Prize Best Flash Talk</h4><div class="caption hidden col-lg-6 text-start" id=caption><h4>2nd Prize Best Flash Talk</h4><p>2nd Symposium for Spanish-Speaking Students</p></div><span style=background-image:url(/images/sections/achievements/flashtalk_hu_5709965b2f901028.jpeg) class=d-none id=SmallImage active=true></span>
<span style=background-image:url(/images/sections/achievements/flashtalk_hu_7f2272d413c73b9f.jpeg) class=d-none id=LargeImage></span></div><div class="achievement-entry text-center" style=background-image:url(/images/sections/achievements/accesit_phd_hu_1e7cbb375e6660b1.jpeg)><i class="fa-solid fa-xmark hidden"></i>
<i class="fa-solid fa-magnifying-glass-plus" id=enlarge-icon></i><h4 class=title id=achievement-title>Finalist Best PhD Thesis</h4><div class="caption hidden col-lg-6 text-start" id=caption><h4>Finalist Best PhD Thesis</h4><p>Spanish Society for Bioinformatics and Computational Biology.</p></div><span style=background-image:url(/images/sections/achievements/accesit_phd_hu_1e7cbb375e6660b1.jpeg) class=d-none id=SmallImage active=true></span>
<span style=background-image:url(/images/sections/achievements/accesit_phd_hu_b99fbcca0937ab6d.jpeg) class=d-none id=LargeImage></span></div><div class="achievement-entry text-center" style=background-image:url(/images/sections/achievements/bestposter_hu_96d075fa1764fe7c.jpeg)><i class="fa-solid fa-xmark hidden"></i>
<i class="fa-solid fa-magnifying-glass-plus" id=enlarge-icon></i><h4 class=title id=achievement-title>Best Poster Prize</h4><div class="caption hidden col-lg-6 text-start" id=caption><h4>Best Poster Prize</h4><p>2nd Congress of the Spanish Society for Bioinformatics and Computational Biology.</p></div><span style=background-image:url(/images/sections/achievements/bestposter_hu_96d075fa1764fe7c.jpeg) class=d-none id=SmallImage active=true></span>
<span style=background-image:url(/images/sections/achievements/bestposter_hu_13bb0e28c9445a21.jpeg) class=d-none id=LargeImage></span></div></div></div></div><div class="container-fluid section-holder d-flex bg-primary"><div class="container-fluid anchor pb-5 publications-section" id=publications><h1 class=text-center><span id=publications></span>Publications</h1><div class="container ms-auto text-center"><div class="btn-group flex-wrap" role=pub-group id=publication-filter-buttons><button type=button class="btn btn-dark pub-filtr-control" data-filter=pub-all>
All
</button>
<button type=button class="btn btn-dark pub-filtr-control" data-filter="pub-First author">
First author
</button>
<button type=button class="btn btn-dark pub-filtr-control" data-filter=pub-Collaboration>
Collaboration
</button>
<button type=button class="btn btn-dark pub-filtr-control" data-filter="pub-Corresponding author">
Corresponding author</button></div></div><div class="container filtr-publications"><div class=row id=publication-card-holder><div class="col-12 p-2 pub-filtr-item" data-category=pub-all,pub-Collaboration><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">The geroprotectors trametinib and rapamycin combine additively to extend mouse healthspan and lifespan</h5><div class=sub-title><span><a href=https://www.nature.com/articles/s43587-025-00876-4>Nature Aging</a>
</span><span class=ms-2>28 May 2025</span></div><div class=authors><span class=me-2><a>Gkioni, L</a></span>
<span class=me-2><a>Nespital, T</a></span>
<span class=me-2><a>Baghdadi, M</a></span>
<span class=me-2><a>Monzó, C</a></span>
<span class=me-2><a>Bali, J</a></span>
<span class=me-2><a>Nassr, T</a></span>
<span class=me-2><a>Cremer, AL</a></span>
<span class=me-2><a>Beyer, A</a></span>
<span class=me-2><a>Deelen, J</a></span>
<span class=me-2><a>Backes, H</a></span>
<span class=me-2><a>Grönke, S</a></span>
<span class=me-2><a>Partridge, L</a></span></div></div><div class=card-body><p>Suppression of the insulin–IGF–mTORC1–Ras network ameliorates aging in animals. Many drugs have targets in the network because of its roles in cancer and metabolic disease and are candidates for repurposing as geroprotectors. Rapamycin, an established geroprotective drug, blocks mTORC1 signaling, and trametinib inhibits the Ras–MEK–ERK pathway. In this study, we assessed survival and health of male and female mice treated with trametinib, rapamycin or their combination. We show here that trametinib treatment extended lifespan in both sexes and that its combination with rapamycin was additive. Combination treatment reduced liver tumors in both sexes and spleen tumors in male mice, blocked the age-related increase in brain glucose uptake and strongly reduced inflammation in brain, kidney, spleen and muscle and circulating levels of pro-inflammatory cytokines. We conclude that trametinib is a geroprotector in mice and that its combination with rapamycin is more effective than either drug alone, making the combination a candidate for repurposing as a gerotherapy in humans.</p></div><div class=card-footer><div class=tags><span class="btn badge btn-info ms-1 p-2">Aging
</span><span class="btn badge btn-info ms-1 p-2">Statistics
</span><span class="btn badge btn-info ms-1 p-2">Data Analysis
</span><span class="btn badge btn-info ms-1 p-2">Proteomics</span></div><div class=details-btn><a class="btn btn-outline-info ms-1 ps-2 mb-2" href=https://www.nature.com/articles/s43587-025-00876-4 target=_blank rel=noopener role=button>Details</a></div></div></div></div><div class="col-12 p-2 pub-filtr-item" data-category='pub-all,pub-First author,pub-Corresponding author'><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Transcriptomics in the era of long-read sequencing</h5><div class=sub-title><span><a href=https://www.nature.com/articles/s41576-025-00828-z>Nature Reviews Genetics</a>
</span><span class=ms-2>28 March 2025</span></div><div class=authors><span class=me-2><a>Monzó, C</a></span>
<span class=me-2><a>Liu, T</a></span>
<span class=me-2><a>Conesa, A</a></span></div></div><div class=card-body><p>Transcriptome sequencing revolutionized the analysis of gene expression, providing an unbiased approach to gene detection and quantification that enabled the discovery of novel isoforms, alternative splicing events and fusion transcripts. However, although short-read sequencing technologies have surpassed the limited dynamic range of previous technologies such as microarrays, they have limitations, for example, in resolving full-length transcripts and complex isoforms. Over the past 5 years, long-read sequencing technologies have matured considerably, with improvements in instrumentation and analytical methods, enabling their application to RNA sequencing (RNA-seq). Benchmarking studies are beginning to identify the strengths and limitations of long-read RNA-seq, although there remains a need for comprehensive resources to guide newcomers through the intricacies of this approach. In this Review, we provide a comprehensive overview of the long-read RNA-seq workflow, from library preparation and sequencing challenges to core data processing, downstream analyses and emerging developments. We present an extensive inventory of experimental and analytical methods and discuss current challenges and prospects.</p></div><div class=card-footer><div class=tags><span class="btn badge btn-info ms-1 p-2">Review
</span><span class="btn badge btn-info ms-1 p-2">Software
</span><span class="btn badge btn-info ms-1 p-2">Long-reads
</span><span class="btn badge btn-info ms-1 p-2">Transcriptomics</span></div><div class=details-btn><a class="btn btn-outline-info ms-1 ps-2 mb-2" href=https://www.nature.com/articles/s41576-025-00828-z target=_blank rel=noopener role=button>Details</a></div></div></div></div><div class="col-12 p-2 pub-filtr-item" data-category='pub-all,pub-First author'><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">MOSim bulk and single-cell multilayer regulatory network simulator</h5><div class=sub-title><span><a href=https://academic.oup.com/bib/article/26/2/bbaf110/8089935>Briefings in Bioinformatics</a>
</span><span class=ms-2>21 March 2025</span></div><div class=authors><span class=me-2><a>Monzó, C</a></span>
<span class=me-2><a>Aguerralde-Martin, M</a></span>
<span class=me-2><a>Martínez-Mira, C</a></span>
<span class=me-2><a>Arzalluz-Luque, A</a></span>
<span class=me-2><a>Conesa, A</a></span>
<span class=me-2><a>Tarazona, S</a></span></div></div><div class=card-body><p>As multi-omics sequencing technologies advance, the need for simulation tools capable of generating realistic and diverse (bulk and single-cell) multi-omics datasets for method testing and benchmarking becomes increasingly important. We present MOSim, an R package that simulates both bulk (via mosim function) and single-cell (via sc_mosim function) multi-omics data. The mosim function generates bulk transcriptomics data (RNA-seq) and additional regulatory omics layers (ATAC-seq, miRNA-seq, ChIP-seq, Methyl-seq, and transcription factors), while sc_mosim simulates single-cell transcriptomics data (scRNA-seq) with scATAC-seq and transcription factors as regulatory layers. The tool supports various experimental designs, including simulation of gene co-expression patterns, biological replicates, and differential expression between conditions. MOSim enables users to generate quantification matrices for each simulated omics data type, capturing the heterogeneity and complexity of bulk and single-cell multi-omics datasets. Furthermore, MOSim provides differentially abundant features within each omics layer and elucidates the active regulatory relationships between regulatory omics and gene expression data at both bulk and single-cell levels. By leveraging MOSim, researchers will be able to generate realistic and customizable bulk and single-cell multi-omics datasets to benchmark and validate analytical methods specifically designed for the integrative analysis of diverse regulatory omics data.</p></div><div class=card-footer><div class=tags><span class="btn badge btn-info ms-1 p-2">Software
</span><span class="btn badge btn-info ms-1 p-2">Single-cell
</span><span class="btn badge btn-info ms-1 p-2">Multi-omics
</span><span class="btn badge btn-info ms-1 p-2">Statistics</span></div></div></div></div><div class="col-12 p-2 pub-filtr-item" data-category=pub-all,pub-Collaboration><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Quality assessment of long read data in multisample lrRNA-seq experiments with SQANTI-reads</h5><div class=sub-title><span><a href=https://genome.cshlp.org/content/35/4/987>Genome Research</a>
</span><span class=ms-2>3 March 2025</span></div><div class=authors><span class=me-2><a>Keil, N</a></span>
<span class=me-2><a>Monzó, C</a></span>
<span class=me-2><a>McIntyre, L</a></span>
<span class=me-2><a>Conesa, A</a></span></div></div><div class=card-body><p>SQANTI-reads leverages SQANTI3, a tool for the analysis of the quality of transcript models, to develop a read-level quality control framework for replicated long-read RNA-seq experiments. The number and distribution of reads, as well as the number and distribution of unique junction chains (transcript splicing patterns), in SQANTI3 structural categories are informative of raw data quality. Multisample visualizations of QC metrics are presented by experimental design factors to identify outliers. We introduce new metrics for (1) the identification of potentially under-annotated genes and putative novel transcripts and for (2) quantifying variation in junction donors and acceptors. We applied SQANTI-reads to two different data sets, a Drosophila developmental experiment and a multiplatform data set from the LRGASP project and demonstrate that the tool effectively reveals the impact of read coverage on data quality, and readily identifies strong and weak splicing sites.</p></div><div class=card-footer><div class=tags><span class="btn badge btn-info ms-1 p-2">Software
</span><span class="btn badge btn-info ms-1 p-2">Long-reads
</span><span class="btn badge btn-info ms-1 p-2">Transcriptomics</span></div><div class=details-btn><a class="btn btn-outline-info ms-1 ps-2 mb-2" href=https://genome.cshlp.org/content/35/4/987 target=_blank rel=noopener role=button>Details</a></div></div></div></div><div class="col-12 p-2 pub-filtr-item" data-category='pub-all,pub-First author'><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Notable challenges posed by long-read sequencing for the study of transcriptional diversity and genome annotation</h5><div class=sub-title><span><a href=https://genome.cshlp.org/content/35/4/583>Genome Research</a>
</span><span class=ms-2>3 March 2025</span></div><div class=authors><span class=me-2><a>Monzó, C</a></span>
<span class=me-2><a>Frankish, A</a></span>
<span class=me-2><a>Conesa, A</a></span></div></div><div class=card-body><p>Long-read sequencing (LRS) technologies have revolutionized transcriptomic research by enabling the comprehensive sequencing of full-length transcripts. Using these technologies, researchers have reported tens of thousands of novel transcripts, even in well-annotated genomes, while developing new algorithms and experimental approaches to handle the noisy data. The Long-read RNA-seq Genome Annotation Assessment Project community effort benchmarked LRS methods in transcriptomics and validated many novel, lowly expressed, often times sample-specific transcripts identified by long reads. These molecules represent deviations of the major transcriptional program that were overlooked by short-read sequencing methods but are now captured by the full-length, single-molecule approach. This Perspective discusses the challenges and opportunities associated with LRS’ capacity to unravel this fraction of the transcriptome, in terms of both transcriptome biology and genome annotation. For transcriptome biology, we need to develop novel experimental and computational methods to effectively differentiate technology errors from rare but real molecules. For genome annotation, we must agree on the strategy to capture molecular variability while still defining reference annotations that are useful for the genomics community.</p></div><div class=card-footer><div class=tags><span class="btn badge btn-info ms-1 p-2">Review
</span><span class="btn badge btn-info ms-1 p-2">Long-reads
</span><span class="btn badge btn-info ms-1 p-2">Transcriptomics</span></div><div class=details-btn><a class="btn btn-outline-info ms-1 ps-2 mb-2" href=https://genome.cshlp.org/content/35/4/583 target=_blank rel=noopener role=button>Details</a></div></div></div></div><div class="col-12 p-2 pub-filtr-item" data-category=pub-all,pub-Collaboration><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Intermittent rapamycin feeding recapitulates some effects of continuous treatment while maintaining lifespan extension</h5><div class=sub-title><span><a href=https://www.sciencedirect.com/science/article/pii/S2212877824000334>Molecular Metabolism</a>
</span><span class=ms-2>21 Febrero 2024</span></div><div class=authors><span class=me-2><a>Baghdadi, M</a></span>
<span class=me-2><a>Nespital, T</a></span>
<span class=me-2><a>Monzó, C</a></span>
<span class=me-2><a>Deelen, J</a></span>
<span class=me-2><a>Grönke, S</a></span>
<span class=me-2><a>Partridge, L</a></span></div></div><div class=card-body><p>Rapamycin, a powerful geroprotective drug, can have detrimental effects when administered chronically. We determined whether intermittent treatment of mice can reduce negative effects while maintaining benefits of chronic treatment. From 6 months of age, male and female C3B6F1 hybrid mice were either continuously fed with 42 mg/kg rapamycin, or intermittently fed by alternating weekly feeding of 42 mg/kg rapamycin food with weekly control feeding. Survival of these mice compared to control animals was measured. Furthermore, longitudinal phenotyping including metabolic (body composition, GTT, ITT, indirect calorimetry) and fitness phenotypes (treadmil, rotarod, electrocardiography and open field) was performed. Organ specific pathology was assessed at 24 months of age. Chronic rapamycin treatment induced glucose intolerance, which was partially ameliorated by intermittent treatment. Chronic and intermittent rapamycin treatments increased lifespan equally in males, while in females chronic treatment resulted in slightly higher survival. The two treatments had equivalent effects on testicular degeneration, heart fibrosis and liver lipidosis. In males, the two treatment regimes led to a similar increase in motor coordination, heart rate and Q-T interval, and reduction in spleen weight, while in females, they equally reduced BAT inflammation and spleen weight and maintained heart rate and Q-T interval. However, other health parameters, including age related pathologies, were better prevented by continuous treatment. Intermittent rapamycin treatment is effective in prolonging lifespan and reduces some side-effects of chronic treatment, but chronic treatment is more beneficial to healthspan.</p></div><div class=card-footer><div class=tags><span class="btn badge btn-info ms-1 p-2">Statistics
</span><span class="btn badge btn-info ms-1 p-2">Data Analysis
</span><span class="btn badge btn-info ms-1 p-2">Proteomics</span></div><div class=details-btn><a class="btn btn-outline-info ms-1 ps-2 mb-2" href=https://www.sciencedirect.com/science/article/pii/S2212877824000334 target=_blank rel=noopener role=button>Details</a></div></div></div></div><div class="col-12 p-2 pub-filtr-item" data-category='pub-all,pub-First author'><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Dietary restriction mitigates the age-associated decline in mouse B cell receptor repertoire diversity</h5><div class=sub-title><span><a href=https://www.cell.com/cell-reports/fulltext/S2211-1247%2823%2900733-7>Cell Reports</a>
</span><span class=ms-2>25 July 2023</span></div><div class=authors><span class=me-2><a>Monzó, C</a></span>
<span class=me-2><a>Gkioni, L</a></span>
<span class=me-2><a>Beyer, A</a></span>
<span class=me-2><a>Valenzano, D</a></span>
<span class=me-2><a>Grönke, S</a></span>
<span class=me-2><a>Partridge, L</a></span></div></div><div class=card-body><p>Aging impairs the capacity to respond to novel antigens, reducing immune protection against pathogens and vaccine efficacy. Dietary restriction (DR) extends life- and health span in diverse animals. However, little is known about the capacity of DR to combat the decline in immune function. Here, we study the changes in B cell receptor (BCR) repertoire during aging in DR and control mice. By sequencing the variable region of the BCR heavy chain in the spleen, we show that DR preserves diversity and attenuates the increase in clonal expansions throughout aging. Remarkably, mice starting DR in mid-life have repertoire diversity and clonal expansion rates indistinguishable from chronic DR mice. In contrast, in the intestine, these traits are unaffected by either age or DR. Reduced within-individual B cell repertoire diversity and increased clonal expansions are correlated with higher morbidity, suggesting a potential contribution of B cell repertoire dynamics to health during aging.</p></div><div class=card-footer><div class=tags><span class="btn badge btn-info ms-1 p-2">Aging
</span><span class="btn badge btn-info ms-1 p-2">Statistics
</span><span class="btn badge btn-info ms-1 p-2">Data Analysis
</span><span class="btn badge btn-info ms-1 p-2">Transcriptomics</span></div><div class=details-btn><a class="btn btn-outline-info ms-1 ps-2 mb-2" href=https://www.cell.com/cell-reports/fulltext/S2211-1247%2823%2900733-7 target=_blank rel=noopener role=button>Details</a></div></div></div></div><div class="col-12 p-2 pub-filtr-item" data-category=pub-all,pub-Collaboration><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Dynamics of Human Anelloviruses in Plasma and Clinical Outcomes Following Kidney Transplantation</h5><div class=sub-title><span><a href=https://journals.lww.com/transplantjournal/fulltext/2023/02000/dynamics_of_human_anelloviruses_in_plasma_and.34.aspx>Transplantation</a>
</span><span class=ms-2>31 August 2022</span></div><div class=authors><span class=me-2><a>Forqué, L</a></span>
<span class=me-2><a>Fernández-Ruiz, M</a></span>
<span class=me-2><a>Albert, E</a></span>
<span class=me-2><a>Giménez, E</a></span>
<span class=me-2><a>Monzó, C</a></span>
<span class=me-2><a>Chaves, J</a></span>
<span class=me-2><a>Redondo, N</a></span>
<span class=me-2><a>Rodríguez-Goncer, I</a></span>
<span class=me-2><a>Ruiz-Merlo, T</a></span>
<span class=me-2><a>Parra, P</a></span>
<span class=me-2><a>Andrés, A</a></span>
<span class=me-2><a>Aguado, JM</a></span>
<span class=me-2><a>Navarro, D</a></span></div></div><div class=card-body><p>Torque teno virus, the major member of the genus Alphatorquevirus , is an emerging biomarker of the net state of immunosuppression after kidney transplantation. Genetic diversity constitutes a main feature of the Anelloviridae family, although its posttransplant dynamics and clinical correlates are largely unknown. The relative abundance of Alphatorquevirus , Betatorquevirus , and Gammatorquevirus genera was investigated by high-throughput sequencing in plasma specimens obtained at various points during the first posttransplant year (n = 91 recipients). Total loads of all members of the Anelloviridae family were also quantified by an &ldquo;in-house&rdquo; polymerase chain reaction assay targeting conserved DNA sequences (n = 195 recipients). In addition to viral kinetics, clinical study outcomes included serious infection, immunosuppression-related adverse event (opportunistic infection and cancer)&rsquo; and acute rejection. Alphatorquevirus DNA was detected in all patients at every point, with an increase from pretransplantation to month 1. A variable proportion of recipients had detectable Betatorquevirus and Gammatorquevirus at lower frequencies. At least 1 change in the predominant genus (mainly as early transition to Alphatorquevirus predominance) was shown in 35.6% of evaluable patients. Total anelloviruses DNA levels increased from baseline to month 1, to peak by month 3 and decrease thereafter, and were higher in patients treated with T-cell depleting agents. There was a significant albeit weak-to-moderate correlation between total anelloviruses and TTV DNA levels. No associations were found between the predominant Anelloviridae genus or total anelloviruses DNA levels and clinical outcomes. Our study provides novel insight into the evolution of the anellome after kidney transplantation.</p></div><div class=card-footer><div class=tags><span class="btn badge btn-info ms-1 p-2">Microbiome
</span><span class="btn badge btn-info ms-1 p-2">Statistics
</span><span class="btn badge btn-info ms-1 p-2">Data Analysis</span></div><div class=details-btn><a class="btn btn-outline-info ms-1 ps-2 mb-2" href=https://journals.lww.com/transplantjournal/fulltext/2023/02000/dynamics_of_human_anelloviruses_in_plasma_and.34.aspx target=_blank rel=noopener role=button>Details</a></div></div></div></div><div class="col-12 p-2 pub-filtr-item" data-category=pub-all,pub-Collaboration><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Benchmarking different approaches for Norovirus genome assembly in metagenome samples</h5><div class=sub-title><span><a href=https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-021-08067-2>BMC Genomics</a>
</span><span class=ms-2>24 November 2021</span></div><div class=authors><span class=me-2><a>Fuentes-Trillo, A</a></span>
<span class=me-2><a>Monzó, C</a></span>
<span class=me-2><a>Manzano, I</a></span>
<span class=me-2><a>Santiso-Bellón, C</a></span>
<span class=me-2><a>da Silva, J</a></span>
<span class=me-2><a>Gozalbo-Rovira, R</a></span>
<span class=me-2><a>García-García, AB</a></span>
<span class=me-2><a>Rodríguez-Díaz, J</a></span>
<span class=me-2><a>Chaves, FJ</a></span></div></div><div class=card-body><p>Genome assembly of viruses with high mutation rates, such as Norovirus and other RNA viruses, or from metagenome samples, poses a challenge for the scientific community due to the coexistence of several viral quasispecies and strains. Furthermore, there is no standard method for obtaining whole-genome sequences in non-related patients. After polyA RNA isolation and sequencing in eight patients with acute gastroenteritis, we evaluated two de Bruijn graph assemblers (SPAdes and MEGAHIT), combined with four different and common pre-assembly strategies, and compared those yielding whole genome Norovirus contigs. Reference-genome guided strategies with both host and target virus did not present any advantages compared to the assembly of non-filtered data in the case of SPAdes, and in the case of MEGAHIT, only host genome filtering presented improvements. MEGAHIT performed better than SPAdes in most samples, reaching complete genome sequences in most of them for all the strategies employed. Read binning with CD-HIT improved assembly when paired with different analysis strategies, and more notably in the case of SPAdes. Not all metagenome assemblies are equal and the choice in the workflow depends on the species studied and the prior steps to analysis. We may need different approaches even for samples treated equally due to the presence of high intra host variability. We tested and compared different workflows for the accurate assembly of Norovirus genomes and established their assembly capacities for this purpose.</p></div><div class=card-footer><div class=tags><span class="btn badge btn-info ms-1 p-2">Microbiome
</span><span class="btn badge btn-info ms-1 p-2">Software
</span><span class="btn badge btn-info ms-1 p-2">Statistics</span></div><div class=details-btn><a class="btn btn-outline-info ms-1 ps-2 mb-2" href=https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-021-08067-2 target=_blank rel=noopener role=button>Details</a></div></div></div></div><div class="col-12 p-2 pub-filtr-item" data-category='pub-all,pub-First author'><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Diversity and dynamic changes of anelloviruses in plasma following allogenic hematopoietic stem cell transplantation</h5><div class=sub-title><span><a href=https://onlinelibrary.wiley.com/doi/abs/10.1002/jmv.26661>Journal of Medical Virology</a>
</span><span class=ms-2>11 November 2021</span></div><div class=authors><span class=me-2><a>Monzó, C</a></span>
<span class=me-2><a>Giménez, E</a></span>
<span class=me-2><a>Albert, E</a></span>
<span class=me-2><a>Fuentes-Trillo, A</a></span>
<span class=me-2><a>Seda, E</a></span>
<span class=me-2><a>Piñana, JL</a></span>
<span class=me-2><a>Hernández Boluda, JC</a></span>
<span class=me-2><a>Solano, C</a></span>
<span class=me-2><a>Chaves, FJ</a></span>
<span class=me-2><a>Navarro, D</a></span></div></div><div class=card-body><p>Monitoring of alphatorquevirus (torque teno virus [TTV]) DNA in plasma may prove to be useful to assess the net state of immune competence following allogeneic hematopoietic stem cell transplantation (allo-HSCT). There are scarce data published on the prevalence of beta (torque teno mini virus [TTMV]) and gammatorqueviruses (torque teno midi virus [TTMDV]) and, in particular, on the dynamics of anelloviruses in allo-HSCT patients. Twenty-five allo-HSCT recipients with available plasma specimens obtained before conditioning and after engraftment were included. Degenerated primers targeting a highly conserved genomic sequence across all anelloviruses were designed for genomic amplification and high-throughput sequencing. Co-detection of TTV, TTMV, and TTMDV both in pre-transplant and post-engraftment plasma specimens was documented in more than two-thirds of patients. The use of quantitative real-time polymerase chain reaction (PCR) assays targeting TTMV and TTMDV in addition to TTV may add value to TTV-specific PCR assays in the inference of the net state of immunosuppresion or immune competence in this clinical setting.</p></div><div class=card-footer><div class=tags><span class="btn badge btn-info ms-1 p-2">Microbiome
</span><span class="btn badge btn-info ms-1 p-2">Statistics
</span><span class="btn badge btn-info ms-1 p-2">Data Analysis</span></div><div class=details-btn><a class="btn btn-outline-info ms-1 ps-2 mb-2" href=https://onlinelibrary.wiley.com/doi/abs/10.1002/jmv.26661 target=_blank rel=noopener role=button>Details</a></div></div></div></div><div class="col-12 p-2 pub-filtr-item" data-category=pub-all,pub-Collaboration><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Immune deficiency in Jacobsen Syndrome, Molecular and phenotypic characterization</h5><div class=sub-title><span><a href=https://www.mdpi.com/2073-4425/12/8/1197>Genes</a>
</span><span class=ms-2>31 July 2021</span></div><div class=authors><span class=me-2><a>Rodríguez-López, R</a></span>
<span class=me-2><a>Gimeno-Ferrer, F</a></span>
<span class=me-2><a>Montesinos, E</a></span>
<span class=me-2><a>Ferrer-Bolufer, I</a></span>
<span class=me-2><a>Guzmán Luján, C</a></span>
<span class=me-2><a>Albuquerque, D</a></span>
<span class=me-2><a>Monzó, C</a></span>
<span class=me-2><a>Ballesteros, V</a></span>
<span class=me-2><a>Pérez-Gramunt, MA</a></span></div></div><div class=card-body><p>Jacobsen syndrome or JBS (OMIM 147791) is a contiguous gene syndrome caused by a deletion affecting the terminal q region of chromosome 11. The phenotype of patients with JBS is a specific syndromic phenotype predominately associated with hematological alterations. Complete and partial JBS are differentiated depending on which functional and causal genes are haploinsufficient in the patient. We describe the case of a 6-year-old Bulgarian boy in which it was possible to identify all of the major signs and symptoms listed by the Online Mendelian Inheritance in Man (OMIM) catalog using the Human Phenotype Ontology (HPO). Extensive blood and marrow tests revealed the existence of thrombocytopenia and leucopenia, specifically due to low levels of T and B cells and low levels of IgM. Genetic analysis using whole-genome single nucleotide polymorphisms (SNPs)/copy number variations (CNVs) microarray hybridization confirmed that the patient had the deletion arr[hg19]11q24.3q25(128,137,532–134,938,470)x1 in heterozygosis. This alteration was considered causal of partial JBS because the essential BSX and NRGN genes were not included, though 30 of the 96 HPO identifiers associated with this OMIM were identified in the patient. The deletion of the FLI-1, ETS1, JAM3 and THYN1 genes was considered to be directly associated with the immunodeficiency exhibited by the patient. Although immunodeficiency is widely accepted as a major sign of JBS, only constipation, bone marrow hypocellularity and recurrent respiratory infections have been included in the HPO as terms used to refer to the immunological defects in JBS. Exhaustive functional analysis and individual monitoring are required and should be mandatory for these patients.</p></div><div class=card-footer><div class=tags><span class="btn badge btn-info ms-1 p-2">Medical Genetics
</span><span class="btn badge btn-info ms-1 p-2">Statistics</span></div><div class=details-btn><a class="btn btn-outline-info ms-1 ps-2 mb-2" href=https://www.mdpi.com/2073-4425/12/8/1197 target=_blank rel=noopener role=button>Details</a></div></div></div></div><div class="col-12 p-2 pub-filtr-item" data-category='pub-all,pub-First author'><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Nearly complete genome sequences of Human Norovirus belonging to several genotypes from Valencia, Spain</h5><div class=sub-title><span><a href=https://journals.asm.org/doi/10.1128/mra.00641-19>Microbiology Resource Announcements</a>
</span><span class=ms-2>22 August 2019</span></div><div class=authors><span class=me-2><a>Monzó, C</a></span>
<span class=me-2><a>Santiso-Bellón, C</a></span>
<span class=me-2><a>Fuentes-Trillo, A</a></span>
<span class=me-2><a>Vila-Vicent, S</a></span>
<span class=me-2><a>da Silva, J</a></span>
<span class=me-2><a>Gozalbo-Rovira, R</a></span>
<span class=me-2><a>Buesa, J</a></span>
<span class=me-2><a>Chaves, FJ</a></span>
<span class=me-2><a>Rodríguez-Díaz, J</a></span></div></div><div class=card-body><p>Human noroviruses are responsible for most nonbacterial acute gastroenteritis cases. The GII.2, GII.4, and GII.17 genotypes of human noroviruses have recently arisen as the most frequent genotypes found in humans worldwide. We report here seven nearly complete genomes of these genotypes from patients with acute gastroenteritis in Valencia, Spain.</p></div><div class=card-footer><div class=tags><span class="btn badge btn-info ms-1 p-2">Microbiome
</span><span class="btn badge btn-info ms-1 p-2">Data Analysis</span></div><div class=details-btn><a class="btn btn-outline-info ms-1 ps-2 mb-2" href=https://journals.asm.org/doi/10.1128/mra.00641-19 target=_blank rel=noopener role=button>Details</a></div></div></div></div><div class="col-12 p-2 pub-filtr-item" data-category=pub-all,pub-Collaboration><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Caracterización molecular de la coexistencia de haploinsuficiencia 18q y duplicación 18p en paciente con fenotipo sindrómico complejo</h5><div class=sub-title><span><a href=https://genotipia.com/revista_gm/gmgcg13-gimeno-ferrer/>Revista de Genética Médica y Genómica</a>
</span><span class=ms-2>25 Abril 2019</span></div><div class=authors><span class=me-2><a>Gimeno-Ferrer, F</a></span>
<span class=me-2><a>Albuquerque, D</a></span>
<span class=me-2><a>Capataz Ledesma, M</a></span>
<span class=me-2><a>Monzó, C</a></span>
<span class=me-2><a>Gervasini, G</a></span>
<span class=me-2><a>Barros Angueira, F</a></span>
<span class=me-2><a>Carbonell, JM</a></span>
<span class=me-2><a>Marcaida Benito, G</a></span>
<span class=me-2><a>Rodríguez-López, R</a></span>
<span class=me-2><a>Galán Gómez, E</a></span></div></div><div class=card-body><p>Since genomic SNPs/CNVs arrays were implemented as a diagnostic tool in clinical settings to search for the cause of idiopathic intellectual disability, chromosomal imbalances have been precisely described as being the cause of many new syndromes, especially when they are associated with multiple congenital anomalies and/or dimorphism. High-density SNPs/CNVs microarray was used to delineate genotype-phenotype correlation in a 2.5 year-old girl who carried a mosaic characterized by a predominant cell line representing 92% which carried a duplication of 12.5 Mb of the 18p11.32p11.21 chromosomal region (chr18:12602631_telomeric) and a deletion and a deletion of 20.3 Mb of the 18q21.32q23 chromosomal region (chr18:57691236_telomeric) and a second minor cell line (8%) presented a ring chromosome, carrying the deletion of 20.3 Mb of the 18q21.32q23 chromosomal region. The microarray analysis identified the genetic causes for the specific phenotype of the patient, whose more evident signs and symptoms were mainly associated to the genomic regions duplicated and in haploinsufficiency. Although the conventional karyotype is currently considered a diagnostic technique of support or second line, it was key to establish the etiology of the alteration that the patient carried. Its performance allowed the correct interpretation of the result of the array, whose characterization facilitated its correlation with the phenotypic features of the patient. From both tests, it was possible to conclude the existence of ring chromosome 18, as well as the percentage of the mosaic lines. The deleterious capabilities of the affected OMIM and dominant genes were evaluated along her period of life. A rigorous clinical following up in that patient included valorous phenotypic data to the array data bases, and will make less difficult to hypothesize about prognosis in other individuals who carry overlapping similar high risk genetic alterations.</p></div><div class=card-footer><div class=tags><span class="btn badge btn-info ms-1 p-2">Medical Genetics</span></div><div class=details-btn><a class="btn btn-outline-info ms-1 ps-2 mb-2" href=https://genotipia.com/revista_gm/gmgcg13-gimeno-ferrer/ target=_blank rel=noopener role=button>Details</a></div></div></div></div><div class="col-12 p-2 pub-filtr-item" data-category=pub-all,pub-Collaboration><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Nearly complete genome sequence of a Human Norovirus GII.P17-GII.17 strain isolated from Brazil in 2015</h5><div class=sub-title><span><a href=https://journals.asm.org/doi/10.1128/mra.01376-18>Microbiome Resource Anouncements</a>
</span><span class=ms-2>31 January 2019</span></div><div class=authors><span class=me-2><a>Santiso-Bellón, C</a></span>
<span class=me-2><a>Fuentes-Trillo, A</a></span>
<span class=me-2><a>da Silva, J</a></span>
<span class=me-2><a>Monzó, C</a></span>
<span class=me-2><a>Vila-Vicent, S</a></span>
<span class=me-2><a>Gozalbo-Rovira, R</a></span>
<span class=me-2><a>Buesa, J</a></span>
<span class=me-2><a>Chaves, FJ</a></span>
<span class=me-2><a>Pereira Miagostovich, M</a></span>
<span class=me-2><a>Rodríguez-Díaz, J</a></span></div></div><div class=card-body><p>Human noroviruses are the most common cause of nonbacterial acute gastroenteritis worldwide. We report here the nearly complete genome sequence (7,551 nucleotides) of a human norovirus GII.P17-GII.17 strain detected in July 2015 in the stool sample from an adult with acute gastroenteritis in Brazil.</p></div><div class=card-footer><div class=tags><span class="btn badge btn-info ms-1 p-2">Microbiome
</span><span class="btn badge btn-info ms-1 p-2">Data Analysis</span></div><div class=details-btn><a class="btn btn-outline-info ms-1 ps-2 mb-2" href=https://journals.asm.org/doi/10.1128/mra.01376-18 target=_blank rel=noopener role=button>Details</a></div></div></div></div><div class="col-12 p-2 pub-filtr-item" data-category='pub-all,pub-First author'><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Book chapter, Secuenciación masiva, del ADN a los datos</h5><div class=sub-title><span><a href=https://www.academia.edu/43257170/GEN%C3%93MICA_EN_MEDICINA>Medigene Press SL.</a>
</span><span class=ms-2>12 December 2017</span></div><div class=authors><span class=me-2><a>Monzó, C</a></span>
<span class=me-2><a>Juanes, JM</a></span>
<span class=me-2><a>Marín-García, P</a></span>
<span class=me-2><a>Chaves, FJ</a></span>
<span class=me-2><a>Arnau, V</a></span>
<span class=me-2><a>García-García, AB</a></span></div></div><div class=card-body><p>The first step in any genetic analysis is DNA collection and processing. Each type of genetic analysis requires specific processing, so the path from DNA to data is not always the same, making it essential to provide appropriate material for each analysis. This chapter reviews the steps from DNA collection to the raw data that will enable subsequent clinical interpretation.</p></div><div class=card-footer><div class=tags><span class="btn badge btn-info ms-1 p-2">Review
</span><span class="btn badge btn-info ms-1 p-2">Medical Genetics</span></div><div class=details-btn><a class="btn btn-outline-info ms-1 ps-2 mb-2" href=https://www.academia.edu/43257170/GEN%C3%93MICA_EN_MEDICINA target=_blank rel=noopener role=button>Details</a></div></div></div></div><div class="col-12 p-2 pub-filtr-item" data-category='pub-all,pub-First author'><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Alström syndrome caused by deletion in ALMS1 gene fixed in a Northern Pakistan recurrent haplotype</h5><div class=sub-title><span><a href=https://mansapublishers.com/ijcr/article/view/403>Indian Journal of Case Reports</a>
</span><span class=ms-2>28 November 2017</span></div><div class=authors><span class=me-2><a>Monzó, C</a></span>
<span class=me-2><a>Gimeno-Ferrer, F</a></span>
<span class=me-2><a>Ferrer García, JC</a></span>
<span class=me-2><a>Amadoz, A</a></span>
<span class=me-2><a>Albuquerque, D</a></span>
<span class=me-2><a>Barros Angueira, F</a></span>
<span class=me-2><a>Marcaida, G</a></span>
<span class=me-2><a>Rodríguez-López, R</a></span></div></div><div class=card-body><p>Reduced genetic variability in isolated populations promotes the prevalence of long contiguous stretches of homozygosity (LCSH) that may carry deleterious mutations, manifesting recessive syndromes such as Alström syndrome (OMIM 203800), caused principally by mutations in exons 8, 10, and 16 and deletions/insertions along the ALMS1 gene. Here, Sanger sequencing of these exons and whole-genome copy-number variants/single-nucleotide polymorphisms (SNPs) microarray were used to characterize ALMS1 gene in a 19-year-old Pakistani female with Alström syndrome. Sequencing did not reveal pathogenic alterations but described a set of homozygous polymorphisms. The microarray revealed these SNPs were included in an 8.24 Mb LCSH in 2p12.2p13 that contained a homozygous deletion including exons 13-16 of ALMS1 gene. Therefore, reduced genetic variability in Pakistani population enhanced the inheritance of the homozygous deletion causing Alström syndrome. The comparison of the deletion with known deletions spanning exons 13-16, described on Middle Eastern patients, suggested a fixation of the deletion in a specific haplotype.</p></div><div class=card-footer><div class=tags><span class="btn badge btn-info ms-1 p-2">Medical Genetics</span></div><div class=details-btn><a class="btn btn-outline-info ms-1 ps-2 mb-2" href=https://mansapublishers.com/ijcr/article/view/403 target=_blank rel=noopener role=button>Details</a></div></div></div></div><div class="col-12 p-2 pub-filtr-item" data-category='pub-all,pub-First author'><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Different distribution of the common HFE variants between western and eastern Spanish populations evidences an independent haplotype gene flow</h5><div class=sub-title><span><a href=https://www.elsevier.es/es-revista-revista-del-laboratorio-clinico-282-articulo-different-distribution-common-hfe-variants-S1888400817300223>Revista del Laboratorio Clínico</a>
</span><span class=ms-2>21 April 2017</span></div><div class=authors><span class=me-2><a>Monzó, C</a></span>
<span class=me-2><a>Ortiz, B</a></span>
<span class=me-2><a>Tomás, C</a></span>
<span class=me-2><a>Collado, C</a></span>
<span class=me-2><a>González-Muñoz, C</a></span>
<span class=me-2><a>Guzmán, C</a></span>
<span class=me-2><a>Valverde, V</a></span>
<span class=me-2><a>Miguel-Cabo, M</a></span>
<span class=me-2><a>Monzó, V</a></span>
<span class=me-2><a>de Lamo, M</a></span>
<span class=me-2><a>Marcaida, G</a></span>
<span class=me-2><a>Gervasini, G</a></span>
<span class=me-2><a>Rodríguez-López, R</a></span></div></div><div class=card-body><p>Hereditary hemochromatosis is a disease responsible for excess blood iron. The hemochromatosis gene has two predominant variants, H63D and C282Y single nucleotide polymorphisms. Our study aims to analyze the diagnostic utility of genotyping the 63 and 282 loci, and examine the geographic distribution of these mutations in Spain. Genotyping was performed on 94 healthy control individuals and 324 patients suspected of hereditary hemochromatosis, and also biochemical test to 313 individuals in the patients group. The comparison of allelic frequencies between East and West of Spain, as well as other countries located at a similar longitude, evidenced a west-east distribution gradient of the C282Y allele. In addition, heterogeneous distribution of the H63D mutation in Spain was observed. Patients who carried the 282YY genotype showed significantly higher biochemical parameters (ferritin>300μg/L, Fe>180μg/L, IST>60%, UIBC>355μg/L and CTFH>370μg/dL), which confirmed the correlation between the mutated homozygous genotype and the associated hemochromatosis phenotype. Our results strengthen the importance of executing genetic tests to increase the efficiency of hereditary hemochromatosis diagnosis, which reveal an interesting variability among geographical regions.</p></div><div class=card-footer><div class=tags><span class="btn badge btn-info ms-1 p-2">Medical Genetics
</span><span class="btn badge btn-info ms-1 p-2">Statistics</span></div><div class=details-btn><a class="btn btn-outline-info ms-1 ps-2 mb-2" href=https://www.elsevier.es/es-revista-revista-del-laboratorio-clinico-282-articulo-different-distribution-common-hfe-variants-S1888400817300223 target=_blank rel=noopener role=button>Details</a></div></div></div></div><div class="col-12 p-2 pub-filtr-item" data-category=pub-all,pub-Collaboration><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Genetic susceptibility to Gilbert's syndrome in a valencian population; efficacy of the fasting test</h5><div class=sub-title><span><a href=https://www.sciencedirect.com/science/article/abs/pii/S0014256516301552>Revista Clínica Española</a>
</span><span class=ms-2>18 November 2016</span></div><div class=authors><span class=me-2><a>Torres, AK</a></span>
<span class=me-2><a>Escartín, N</a></span>
<span class=me-2><a>Monzó, C</a></span>
<span class=me-2><a>Guzmán, C</a></span>
<span class=me-2><a>Ferrer, I</a></span>
<span class=me-2><a>González-Muñoz, C</a></span>
<span class=me-2><a>Peña, P</a></span>
<span class=me-2><a>Monzó, V</a></span>
<span class=me-2><a>Marcaida, G</a></span>
<span class=me-2><a>Rodríguez-López, R</a></span></div></div><div class=card-body><p>To describe the populational distribution of the UGT1A1*28 variant (genetic variant code rs8175347) located in the promotor of the UGT gene and correlate its genotypes with the results of the fasting test, as well as its relationship with the biochemical disorder of Gilbert&rsquo;s syndrome (GS) in a Valencian population. We studied the prevalence of the genotypes (TA)6/6 (TA)6/7 and (TA)7/7 of the deleterious variant rs8175347 in 144 patients with hyperbilirubinemia, 38 of whom had previously undergone the fasting test to diagnose GS, and in 150 control patients. By analysing the genomic region of the TATA box of the UGT1A1 gene promotor using Sanger sequencing, we established the correlation between the rs8175347 genotypes and the fasting test results and with the patients’ biochemical disorders. The rate of heterozygosity of allele (TA)7 in the control population was 32% and increased to 87.59% among the patients with suspected GS. The rate of genotype TA7/7 was 81.94% among the patients with hyperbilirubinemia, compared with 11.33% in the control patients. The fasting test showed a 15.79% rate of false negatives and a 5.26% rate of false positives. The high frequency of allele (TA)7 among the Valencian control population, almost double the 5% reported for European control patients, confirms the high rate of GS reported in the Spanish population, without observing significant differences between the geographical ends of the country. The efficacy and reliability of the fasting test for the diagnosis of GS is questionable.</p></div><div class=card-footer><div class=tags><span class="btn badge btn-info ms-1 p-2">Medical Genetics
</span><span class="btn badge btn-info ms-1 p-2">Statistics</span></div><div class=details-btn><a class="btn btn-outline-info ms-1 ps-2 mb-2" href=https://www.sciencedirect.com/science/article/abs/pii/S0014256516301552 target=_blank rel=noopener role=button>Details</a></div></div></div></div><div class="col-12 p-2 pub-filtr-item" data-category='pub-all,pub-First author'><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Molecular analysis of X cromosome in family with X fragile syndrome</h5><div class=sub-title><span><a href=https://genotipia.com/revista_gm/sindrome-x-fragil-caso-genetico/>Genética Médica y Genómica</a>
</span><span class=ms-2>3 June 2016</span></div><div class=authors><span class=me-2><a>Monzó, C</a></span>
<span class=me-2><a>Guzmán, C</a></span>
<span class=me-2><a>Pérez-Gramunt, MA</a></span>
<span class=me-2><a>Escartín, N</a></span>
<span class=me-2><a>Gil Santiago, C</a></span>
<span class=me-2><a>Gandía Artigues, L</a></span>
<span class=me-2><a>Martínez Castellano, F</a></span>
<span class=me-2><a>Marcaida, G</a></span>
<span class=me-2><a>Rodríguez-López, R</a></span></div></div><div class=card-body><p>Fragile X Syndrome is the most common monogenic cause of intellectual disability. It originates by the expansion of the CGG triplet repeat in the 5’ non-coding region of FMR1 gene. We analyzed aforementioned region of susceptibility in relatives of a patient carrying an expansion in range of mutation, with the aim of identifying other carriers of deleterious alleles and offering genetic counseling. Alleles in range of premutation were found in the mother and sister (maternal half-sister) of the patient, with 106 and 60 repetitions respectively; none showed any phenotypic trait. The probability of inheriting a contracted premutated allele when maternally transmitted is low (approximately 0.76%). Therefore, given the finding of a lower number of repetitions in the offspring of a woman, the possibility of a mosaic allele product of a undetected mutated allele has to be ruled out in first stand; for this the employment of a technique of maximum sensitivity is essential. In our case it was impossible to obtain DNA samples from male parents; hence we proceeded with microsatellite analysis on the X chromosome to assign haplotypes segregating with the corresponding expanded alleles, trying to describe their heritability in the pedigree. We conclude that the premutation carried by the sister of the diagnosed patient, was inherited through her paternal route and therefore comes from a second deleterious allele, different from the one transmitted from her mother to her affected sibling. Haplotype structure identified in both carrier chromosomes, the individuals’ population of origin and the heritability by paternal branches, suggests a shared ancestor in previous generations.</p></div><div class=card-footer><div class=tags><span class="btn badge btn-info ms-1 p-2">Medical Genetics</span></div><div class=details-btn><a class="btn btn-outline-info ms-1 ps-2 mb-2" href=https://genotipia.com/revista_gm/sindrome-x-fragil-caso-genetico/ target=_blank rel=noopener role=button>Details</a></div></div></div></div></div></div></div></div><div class="container-fluid section-holder d-flex bg-secondary"><div class="container-fluid anchor pb-5 accomplishments-section"><h1 class=text-center><span id=accomplishments></span>Courses</h1><div class=container><div class=row id=accomplishment-card-holder><div class="col-md-12 col-lg-6 p-2"><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">AWS Cloud Practitioner Essentials</h5><div class=sub-title><span><a href=https://www.coursera.org title="AWS with Coursera" target=_blank rel=noopener>AWS with Coursera</a></span>
<span class=ms-2>August 2025</span></div></div><div class=card-body><p>Core AWS concepts and the benefits of using AWS Cloud. AWS global infrastructure basics. Key AWS services (compute, networking, databases, storage). Financial aspects of AWS (cost models, billing, pricing, and tools to manage expenses)</p></div><div class=card-footer></div></div></div><div class="col-md-12 col-lg-6 p-2"><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Animal Experimentation</h5><div class=sub-title><span><a href=https://experimentacionanimal.ceb.edu.es/ title="Centro de Estudios Biosanitarios" target=_blank rel=noopener>Centro de Estudios Biosanitarios</a></span>
<span class=ms-2>July 2025</span></div></div><div class=card-body><p>Certification course for Function D, experimental and process design, recognized by the Spanish ministry of science.</p></div><div class=card-footer></div></div></div><div class="col-md-12 col-lg-6 p-2"><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Focus on Peer review</h5><div class=sub-title><span><a href=https://masterclasses.nature.com/focus-on-peer-review-online-course/16605550 title="Springer Nature" target=_blank rel=noopener>Springer Nature</a></span>
<span class=ms-2>February 2024</span></div></div><div class=card-body><p>Training course to teach researchers the foundations of good peer review.</p></div><div class=card-footer></div></div></div><div class="col-md-12 col-lg-6 p-2"><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Getting Funded, Writing an Individual Fellowship Application</h5><div class=sub-title><span><a href=http://www.bioscript.de/index.html title=BioScript target=_blank rel=noopener>BioScript</a></span>
<span class=ms-2>May 2022</span></div></div><div class=card-body><p>Training in drafting and writing a fellowship proposal.</p></div><div class=card-footer></div></div></div><div class="col-md-12 col-lg-6 p-2"><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Career Mentoring and Management, Applying for Jobs on the Wider Market</h5><div class=sub-title><span><a href=http://www.bioscript.de/index.html title=BioScript target=_blank rel=noopener>BioScript</a></span>
<span class=ms-2>February 2022</span></div></div><div class=card-body><p>Training on identifying job offers in industry, CV and interview preparation.</p></div><div class=card-footer></div></div></div><div class="col-md-12 col-lg-6 p-2"><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Genomics in Rare Diseases</h5><div class=sub-title><span><a href=https://campus.genotipia.com/cursos-genetica-y-genomica/ title="Genotipia and Illumina" target=_blank rel=noopener>Genotipia and Illumina</a></span>
<span class=ms-2>December 2021</span></div></div><div class=card-body><p>Training in improving genetic dianosis of patients with rare diseases.</p></div><div class=card-footer></div></div></div><div class="col-md-12 col-lg-6 p-2"><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Designing and Presenting a Poster</h5><div class=sub-title><span><a href=http://www.bioscript.de/index.html title=BioScript target=_blank rel=noopener>BioScript</a></span>
<span class=ms-2>October 2021</span></div></div><div class=card-body><p>Training on efficient poster design and presentation.</p></div><div class=card-footer></div></div></div><div class="col-md-12 col-lg-6 p-2"><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Advanced Scientific Writing</h5><div class=sub-title><span><a href=http://www.bioscript.de/index.html title=BioScript target=_blank rel=noopener>BioScript</a></span>
<span class=ms-2>July 2021</span></div></div><div class=card-body><p>Advanced training on drafting and writing scientific articles.</p></div><div class=card-footer></div></div></div><div class="col-md-12 col-lg-6 p-2"><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Machine Learning and Big Data for Bioinformatics</h5><div class=sub-title><span><a href="https://abierta.ugr.es/course/view.php?id=97" title="University of Granada" target=_blank rel=noopener>University of Granada</a></span>
<span class=ms-2>April 2021</span></div></div><div class=card-body><p>Course on machine learning methods for bioinformatics, including supervised and unsupervised methods.</p></div><div class=card-footer></div></div></div><div class="col-md-12 col-lg-6 p-2"><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Good Scientific Practice</h5><div class=sub-title><span><a href=https://www.age.mpg.de/en title="Max Planck Institute" target=_blank rel=noopener>Max Planck Institute</a></span>
<span class=ms-2>March 2019</span></div></div><div class=card-body><p>Course on good scientific practices and cases discussion.</p></div><div class=card-footer></div></div></div><div class="col-md-12 col-lg-6 p-2"><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Algorithms for DNA Sequencing</h5><div class=sub-title><span><a href=https://www.coursera.org title=Coursera target=_blank rel=noopener>Coursera</a></span>
<span class=ms-2>September 2018</span></div></div><div class=card-body><p>Computational methods, algorithms and data structures for analyzing DNA sequencing data.</p></div><div class=card-footer></div></div></div><div class="col-md-12 col-lg-6 p-2"><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Bacterial Genomes, from DNA to protein using bioinformatics</h5><div class=sub-title><span><a href=https://coursesandconferences.wellcomeconnectingscience.org/event/bacterial-genomes-from-dna-to-protein-function-using-bioinformatics-20201012/ title="Welcome Genome Campus on Future Learn" target=_blank rel=noopener>Welcome Genome Campus on Future Learn</a></span>
<span class=ms-2>September 2018</span></div></div><div class=card-body><p>Training to use bioinformatics to explore DNA sequences and protein functions, to evaluate virulence.</p></div><div class=card-footer></div></div></div><div class="col-md-12 col-lg-6 p-2"><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Genomic Medicine in Oncology and it's clinical applications</h5><div class=sub-title><span><a href=https://campus.genotipia.com/formacion/curso-medicina-genomica-en-oncologia/ title="Institute of Genomic Medicine and Medical Genetics Journal" target=_blank rel=noopener>Institute of Genomic Medicine and Medical Genetics Journal</a></span>
<span class=ms-2>May 2018</span></div></div><div class=card-body><p>Training course on using genomics for diagnosis and cancer treatment.</p></div><div class=card-footer></div></div></div><div class="col-md-12 col-lg-6 p-2"><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Networks & Pathways</h5><div class=sub-title><span><a href=https://www.ebi.ac.uk/training/materials/summer-school-materials/group-projects/networks-and-pathways/ title="European Bioinformatics Institute" target=_blank rel=noopener>European Bioinformatics Institute</a></span>
<span class=ms-2>May 2018</span></div></div><div class=card-body><p>Training on exploring biological functions through network analysis.</p></div><div class=card-footer></div></div></div><div class="col-md-12 col-lg-6 p-2"><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Command Line Tools for Genomic Data Science</h5><div class=sub-title><span><a href=https://www.coursera.org title=Coursera target=_blank rel=noopener>Coursera</a></span>
<span class=ms-2>December 2017</span></div></div><div class=card-body><p>Training on using tools such as samtools, bedtools, bwa, vep etc for genomic data analysis.</p></div><div class=card-footer></div></div></div><div class="col-md-12 col-lg-6 p-2"><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Python for Genomic Data Analysis</h5><div class=sub-title><span><a href=https://www.coursera.org title=Coursera target=_blank rel=noopener>Coursera</a></span>
<span class=ms-2>March 2017</span></div></div><div class=card-body><p>Training on using python classes and packages for genomic data analysis.</p></div><div class=card-footer></div></div></div><div class="col-md-12 col-lg-6 p-2"><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">International Course on Genomic Data Analysis, Genomics and Transcriptomics</h5><div class=sub-title><span><a href=https://www.cipf.es/ title="Príncipe Felipe Investigation Centre" target=_blank rel=noopener>Príncipe Felipe Investigation Centre</a></span>
<span class=ms-2>March 2016</span></div></div><div class=card-body><p>Introductory training on using tools for genomic and transcriptomic data analysis.</p></div><div class=card-footer></div></div></div><div class="col-md-12 col-lg-6 p-2"><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Introductory course on identifying sharks and fish</h5><div class=sub-title><span><a href=https://www.uv.es/ title="University of Valencia" target=_blank rel=noopener>University of Valencia</a></span>
<span class=ms-2>April 2014</span></div></div><div class=card-body><p>Theoretical and practical course on identifying and classifying sharks and fish.</p></div><div class=card-footer></div></div></div><div class="col-md-12 col-lg-6 p-2"><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">III Workshop on Morphometry and Shark Dissection</h5><div class=sub-title><span><a href=https://www.uv.es/ title="University of Valencia" target=_blank rel=noopener>University of Valencia</a></span>
<span class=ms-2>February 2014</span></div></div><div class=card-body><p>Theoretical and practical course on correct dissection of sharks.</p></div><div class=card-footer></div></div></div><div class="col-md-12 col-lg-6 p-2"><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Introduction to the identification of marine invertebrates of Mediterranean waters</h5><div class=sub-title><span><a href=https://www.uv.es/ title="University of Valencia" target=_blank rel=noopener>University of Valencia</a></span>
<span class=ms-2>December 2021</span></div></div><div class=card-body><p>Theoretical course on identification and classification of marine invertebrates, focused on cnidaria.</p></div><div class=card-footer></div></div></div></div></div></div></div><div class="container-fluid section-holder d-flex bg-primary"><div class="container-fluid anchor pb-5 projects-section" id=outreach><h1 class=text-center><span id=outreach></span>Outreach</h1><div class="container ms-auto text-center"><div class="btn-group flex-wrap" role=group id=project-filter-buttons><button type=button class="btn btn-dark project-filtr-control" data-filter=all>
All
</button>
<button type=button class="btn btn-dark project-filtr-control" data-filter=general-public>
General Public
</button>
<button type=button class="btn btn-dark project-filtr-control" data-filter=specialized-community>
Specialized Community</button></div></div><div class="container filtr-projects"><div class=row id=project-card-holder><div class="col-sm-12 col-md-6 col-lg-4 p-2 filtr-item" data-category='all, specialized-community'><div class="card mt-1"><div class=card><a href=https://www.earlham.ac.uk/events/long-reads-bioinformatics-summer-school target=_blank rel=noopener><div class=card-head><img class=card-img-top src=/images/sections/outreach/summer_hu_b46a7720845284f4.jpeg alt="Long-reads Bioinformatics Summer School"></div><div class=card-header><div><div class=d-flex><h5 class="card-title mb-0">Long-reads Bioinformatics Summer School</h5></div><div class=sub-title><span>Lecturer and Organizer</span>
<span>July 2025</span></div></div></div></a><div class="card-body text-justify pt-1 pb-1"><p>Introductiory session on study design, sequencing technologies and ground truth data.</p><div class=project-card-footer><div class=project-tags-holder><span class="badge btn-info">specialized-community</span></div><div class=project-btn-holder><span><a class="btn btn-outline-info btn-sm" href=https://www.earlham.ac.uk/events/long-reads-bioinformatics-summer-school target=#>Details</a></span></div></div></div></div></div></div><div class="col-sm-12 col-md-6 col-lg-4 p-2 filtr-item" data-category='all, general-public,specialized-community'><div class="card mt-1"><div class=card><a href=https://www.instagram.com/p/DKy_MvTqi-6/ target=_blank rel=noopener><div class=card-head><img class=card-img-top src=/images/sections/outreach/rsgmosim_hu_b8adecaa16ee1285.jpeg alt="News post about my recent publication on multi-omics."></div><div class=card-header><div><div class=d-flex><h5 class="card-title mb-0">News post about my recent publication on multi-omics.</h5></div><div class=sub-title><span>News</span>
<span>June 2025</span></div></div></div></a><div class="card-body text-justify pt-1 pb-1"><p>Post by the Spanish Regional Student Group in bioinformatics about my latest published work.</p><div class=project-card-footer><div class=project-tags-holder><span class="badge btn-info">general-public
</span><span class="badge btn-info">specialized-community</span></div><div class=project-btn-holder><span><a class="btn btn-outline-info btn-sm" href=https://www.instagram.com/p/DKy_MvTqi-6/ target=#>Details</a></span></div></div></div></div></div></div><div class="col-sm-12 col-md-6 col-lg-4 p-2 filtr-item" data-category='all, general-public'><div class="card mt-1"><div class=card><a href=https://delegacion.comunitatvalenciana.csic.es/el-csic-analiza-una-nueva-tecnica-de-secuenciacion-genica-que-ayuda-a-descifrar-enfermedades-como-el-cancer/ target=_blank rel=noopener><div class=card-head><img class=card-img-top src=/images/sections/outreach/nrg_hu_f56c7868ba420d8a.jpeg alt="Interview and news publication about my recent work."></div><div class=card-header><div><div class=d-flex><h5 class="card-title mb-0">Interview and news publication about my recent work.</h5></div><div class=sub-title><span>Interview</span>
<span>May 2025</span></div></div></div></a><div class="card-body text-justify pt-1 pb-1"><p>Interview about the relevance of my recent publication in Nature Reviews Genetics.</p><div class=project-card-footer><div class=project-tags-holder><span class="badge btn-info">general-public</span></div><div class=project-btn-holder><span><a class="btn btn-outline-info btn-sm" href=https://delegacion.comunitatvalenciana.csic.es/el-csic-analiza-una-nueva-tecnica-de-secuenciacion-genica-que-ayuda-a-descifrar-enfermedades-como-el-cancer/ target=#>Details</a></span></div></div></div></div></div></div><div class="col-sm-12 col-md-6 col-lg-4 p-2 filtr-item" data-category='all, specialized-community'><div class="card mt-1"><div class=card><a href=https://conexion-microbioma.csic.es/evento/conexion-microbioma-celebra-su-1a-asamblea-general/ target=_blank rel=noopener><div class=card-head><img class=card-img-top src=/images/sections/achievements/conmicro_hu_6480f93a3e38b534.jpg alt="1st Assembly of the CSIC Connection on Microbiology."></div><div class=card-header><div><div class=d-flex><h5 class="card-title mb-0">1st Assembly of the CSIC Connection on Microbiology.</h5></div><div class=sub-title><span>Invited talk</span>
<span>May 2025</span></div></div></div></a><div class="card-body text-justify pt-1 pb-1"><p>Talk about the activities of the Junior Bioinformatics and Computational Biology Hub at CSIC.</p><div class=project-card-footer><div class=project-tags-holder><span class="badge btn-info">specialized-community</span></div><div class=project-btn-holder><span><a class="btn btn-outline-info btn-sm" href=https://conexion-microbioma.csic.es/evento/conexion-microbioma-celebra-su-1a-asamblea-general/ target=#>Details</a></span></div></div></div></div></div></div><div class="col-sm-12 col-md-6 col-lg-4 p-2 filtr-item" data-category='all, general-public,specialized-community'><div class="card mt-1"><div class=card><a href=https://www.instagram.com/p/DIgskcxu7Fk/ target=_blank rel=noopener><div class=card-head><img class=card-img-top src=/images/sections/outreach/rsgtd_hu_9075a04125ee1706.jpeg alt="News post about my recent publication on long-reads."></div><div class=card-header><div><div class=d-flex><h5 class="card-title mb-0">News post about my recent publication on long-reads.</h5></div><div class=sub-title><span>News</span>
<span>May 2025</span></div></div></div></a><div class="card-body text-justify pt-1 pb-1"><p>Post by the Spanish Regional Student Group in bioinformatics about my latest published work.</p><div class=project-card-footer><div class=project-tags-holder><span class="badge btn-info">general-public
</span><span class="badge btn-info">specialized-community</span></div><div class=project-btn-holder><span><a class="btn btn-outline-info btn-sm" href=https://www.instagram.com/p/DIgskcxu7Fk/ target=#>Details</a></span></div></div></div></div></div></div><div class="col-sm-12 col-md-6 col-lg-4 p-2 filtr-item" data-category='all, specialized-community'><div class="card mt-1"><div class=card><a href=https://seh2bioinfo.netlify.app/ target=_blank rel=noopener><div class=card-head><img class=card-img-top src=/images/sections/outreach/hispanohablantes_hu_a269d77c2567f9e2.jpeg alt="2nd Symposium for Spanish-Speaking Students in Bioinformatics and Computational Biology"></div><div class=card-header><div><div class=d-flex><h5 class="card-title mb-0">2nd Symposium for Spanish-Speaking Students in Bioinformatics and Computational Biology</h5></div><div class=sub-title><span>Flash talk</span>
<span>April 2025</span></div></div></div></a><div class="card-body text-justify pt-1 pb-1"><p>Flash talk about my project on transcript divergency in aging using long-reads sequencing data.</p><div class=project-card-footer><div class=project-tags-holder><span class="badge btn-info">specialized-community</span></div><div class=project-btn-holder><span><a class="btn btn-outline-info btn-sm" href=https://seh2bioinfo.netlify.app/ target=#>Details</a></span></div></div></div></div></div></div><div class="col-sm-12 col-md-6 col-lg-4 p-2 filtr-item" data-category='all, specialized-community'><div class="card mt-1"><div class=card><a href=https://conexion-bcb.csic.es/evento/2a-asamblea-general/ target=_blank rel=noopener><div class=card-head><img class=card-img-top src=/images/sections/achievements/bcb_hu_9b480723f0bd41d3.jpeg alt="2nd Assembly of the CSIC Connection on Bioinformatics and Computational Biology (BCB Hub)"></div><div class=card-header><div><div class=d-flex><h5 class="card-title mb-0">2nd Assembly of the CSIC Connection on Bioinformatics and Computational Biology (BCB Hub)</h5></div><div class=sub-title><span>Invited talk and round table participant</span>
<span>March 2025</span></div></div></div></a><div class="card-body text-justify pt-1 pb-1"><p>Talk about the activities of the Junior Bioinformatics and Computational Biology Hub at CSIC.</p><div class=project-card-footer><div class=project-tags-holder><span class="badge btn-info">specialized-community</span></div><div class=project-btn-holder><span><a class="btn btn-outline-info btn-sm" href=https://conexion-bcb.csic.es/evento/2a-asamblea-general/ target=#>Details</a></span></div></div></div></div></div></div><div class="col-sm-12 col-md-6 col-lg-4 p-2 filtr-item" data-category='all, specialized-community'><div class="card mt-1"><div class=card><a href="https://www.youtube.com/watch?v=IVBjCTCs3SY&amp;t=2219s" target=_blank rel=noopener><div class=card-head><img class=card-img-top src=/images/sections/achievements/sebibc_hu_6e6597174cbaf450.jpeg alt="Seminar for the Spanish Society of Computational Biology."></div><div class=card-header><div><div class=d-flex><h5 class="card-title mb-0">Seminar for the Spanish Society of Computational Biology.</h5></div><div class=sub-title><span>Invited talk</span>
<span>February 2025</span></div></div></div></a><div class="card-body text-justify pt-1 pb-1"><p>Seminar about my PhD work, finalist to the &ldquo;Best PhD thesis&rdquo;.</p><div class=project-card-footer><div class=project-tags-holder><span class="badge btn-info">specialized-community</span></div><div class=project-btn-holder><span><a class="btn btn-outline-info btn-sm" href="https://www.youtube.com/watch?v=IVBjCTCs3SY&amp;t=2219s" target=#>Details</a></span></div></div></div></div></div></div><div class="col-sm-12 col-md-6 col-lg-4 p-2 filtr-item" data-category='all, general-public'><div class="card mt-1"><div class=card><a href=https://lanochedelainvestigacion.com/researchers-go-to-school/ target=_blank rel=noopener><div class=card-head><img class=card-img-top src=/images/sections/outreach/2025_ResearchersGoToSchool_LuisAmigo_hu_9d6bb8ac121af818.jpeg alt="Researchers go to School"></div><div class=card-header><div><div class=d-flex><h5 class="card-title mb-0">Researchers go to School</h5></div><div class=sub-title><span>Lecturer</span>
<span>February 2025</span></div></div></div></a><div class="card-body text-justify pt-1 pb-1"><p>Talk about women in science to senior high-school students at the Luis Amigó school in Paterna, Valencia.</p><div class=project-card-footer><div class=project-tags-holder><span class="badge btn-info">general-public</span></div><div class=project-btn-holder><span><a class="btn btn-outline-info btn-sm" href=https://lanochedelainvestigacion.com/researchers-go-to-school/ target=#>Details</a></span></div></div></div></div></div></div><div class="col-sm-12 col-md-6 col-lg-4 p-2 filtr-item" data-category='all, general-public'><div class="card mt-1"><div class=card><a href="https://www.youtube.com/watch?v=RqeqHk2V7Kw" target=_blank rel=noopener><div class=card-head><img class=card-img-top src=/images/sections/outreach/bcb11_hu_faab8a33a95bef7a.jpeg alt="BCB Hub CSIC celebrates the day of women in science"></div><div class=card-header><div><div class=d-flex><h5 class="card-title mb-0">BCB Hub CSIC celebrates the day of women in science</h5></div><div class=sub-title><span>Invited talk</span>
<span>February 2025</span></div></div></div></a><div class="card-body text-justify pt-1 pb-1"><p>Introduce myself and my research topic.</p><div class=project-card-footer><div class=project-tags-holder><span class="badge btn-info">general-public</span></div><div class=project-btn-holder><span><a class="btn btn-outline-info btn-sm" href="https://www.youtube.com/watch?v=RqeqHk2V7Kw" target=#>Details</a></span></div></div></div></div></div></div><div class="col-sm-12 col-md-6 col-lg-4 p-2 filtr-item" data-category='all, general-public'><div class="card mt-1"><div class=card><a href=https://portal.edu.gva.es/ieslacanaldenavarres/ target=_blank rel=noopener><div class=card-head><img class=card-img-top src=/images/sections/outreach/canals_hu_5bfa5071eebfa25c.jpeg alt="Day of women in science"></div><div class=card-header><div><div class=d-flex><h5 class="card-title mb-0">Day of women in science</h5></div><div class=sub-title><span>Lecturer</span>
<span>February 2025</span></div></div></div></a><div class="card-body text-justify pt-1 pb-1"><p>Talk about women in science to junior high-school students at the school in La Canal de Navarrés, Valencia.</p><div class=project-card-footer><div class=project-tags-holder><span class="badge btn-info">general-public</span></div><div class=project-btn-holder><span><a class="btn btn-outline-info btn-sm" href=https://portal.edu.gva.es/ieslacanaldenavarres/ target=#>Details</a></span></div></div></div></div></div></div><div class="col-sm-12 col-md-6 col-lg-4 p-2 filtr-item" data-category='all, general-public'><div class="card mt-1"><div class=card><a href="https://www.youtube.com/watch?v=NsPFADgQLpo" target=_blank rel=noopener><div class=card-head><img class=card-img-top src=/images/sections/outreach/bcbaun_hu_c23c0ca1e80ac411.jpeg alt="BCB Hub researcher introduction series"></div><div class=card-header><div><div class=d-flex><h5 class="card-title mb-0">BCB Hub researcher introduction series</h5></div><div class=sub-title><span>Invited talk</span>
<span>January 2025</span></div></div></div></a><div class="card-body text-justify pt-1 pb-1"><p>Introduce myself and my research topic.</p><div class=project-card-footer><div class=project-tags-holder><span class="badge btn-info">general-public</span></div><div class=project-btn-holder><span><a class="btn btn-outline-info btn-sm" href="https://www.youtube.com/watch?v=NsPFADgQLpo" target=#>Details</a></span></div></div></div></div></div></div><div class="col-sm-12 col-md-6 col-lg-4 p-2 filtr-item" data-category='all, specialized-community'><div class="card mt-1"><div class=card><a href=https://www.lrua2024.se/wp-content/uploads/sites/57/2024/10/LRUA24_workshop-program_print.pdf target=_blank rel=noopener><div class=card-head><img class=card-img-top src=/images/sections/outreach/lrua_hu_67835aa4b3616ff2.jpeg alt="Long-read transcriptomics, workflow and applications"></div><div class=card-header><div><div class=d-flex><h5 class="card-title mb-0">Long-read transcriptomics, workflow and applications</h5></div><div class=sub-title><span>Workshop organizer</span>
<span>October 2024</span></div></div></div></a><div class="card-body text-justify pt-1 pb-1"><p>Organization of a long-reads bioinformatics workshop, from library preparation to data analysis and interpretation.</p><div class=project-card-footer><div class=project-tags-holder><span class="badge btn-info">specialized-community</span></div><div class=project-btn-holder><span><a class="btn btn-outline-info btn-sm" href=https://www.lrua2024.se/wp-content/uploads/sites/57/2024/10/LRUA24_workshop-program_print.pdf target=#>Details</a></span></div></div></div></div></div></div><div class="col-sm-12 col-md-6 col-lg-4 p-2 filtr-item" data-category='all, specialized-community'><div class="card mt-1"><div class=card><a href=https://congresosebibc.com/ target=_blank rel=noopener><div class=card-head><img class=card-img-top src=/images/sections/outreach/flash_hu_17df467aaca35e23.jpeg alt="1st National Congress of the Spanish Society for Bioinformatics and Computational Biology"></div><div class=card-header><div><div class=d-flex><h5 class="card-title mb-0">1st National Congress of the Spanish Society for Bioinformatics and Computational Biology</h5></div><div class=sub-title><span>Flash talk</span>
<span>October 2024</span></div></div></div></a><div class="card-body text-justify pt-1 pb-1"><p>Flash talk about my published work MOSim, a bulk and single-cell multi-omics data simulator.</p><div class=project-card-footer><div class=project-tags-holder><span class="badge btn-info">specialized-community</span></div><div class=project-btn-holder><span><a class="btn btn-outline-info btn-sm" href=https://congresosebibc.com/ target=#>Details</a></span></div></div></div></div></div></div><div class="col-sm-12 col-md-6 col-lg-4 p-2 filtr-item" data-category='all, specialized-community'><div class="card mt-1"><div class=card><a href=https://conexion-bcb.csic.es/taller-csic4csic-herramientas-y-servicios-bioinformaticos/ target=_blank rel=noopener><div class=card-head><img class=card-img-top src=/images/sections/outreach/csic4csic_hu_b9914cbeaed94c9e.jpg alt='Workshop on tools and services in computational biology and bioinformatics "CSIC4CSIC"'></div><div class=card-header><div><div class=d-flex><h5 class="card-title mb-0">Workshop on tools and services in computational biology and bioinformatics "CSIC4CSIC"</h5></div><div class=sub-title><span>Lecturer</span>
<span>June 2024</span></div></div></div></a><div class="card-body text-justify pt-1 pb-1"><p>Lecture about the functional iso-transcriptomics pipeline (Bioinformatics).</p><div class=project-card-footer><div class=project-tags-holder><span class="badge btn-info">specialized-community</span></div><div class=project-btn-holder><span><a class="btn btn-outline-info btn-sm" href=https://conexion-bcb.csic.es/taller-csic4csic-herramientas-y-servicios-bioinformaticos/ target=#>Details</a></span></div></div></div></div></div></div><div class="col-sm-12 col-md-6 col-lg-4 p-2 filtr-item" data-category='all, specialized-community'><div class="card mt-1"><div class=card><a href="https://www.youtube.com/watch?v=MAunk99-EZs&amp;t=9s" target=_blank rel=noopener><div class=card-head><img class=card-img-top src=/images/sections/achievements/immcantantion_hu_1a49a915fcc72f86.jpeg alt="The 2024 Immcantantion Users Group Meeting."></div><div class=card-header><div><div class=d-flex><h5 class="card-title mb-0">The 2024 Immcantantion Users Group Meeting.</h5></div><div class=sub-title><span>Invited talk</span>
<span>January 2024</span></div></div></div></a><div class="card-body text-justify pt-1 pb-1"><p>Talk about my published work on B-cells during aging and under anti-aging interventions.</p><div class=project-card-footer><div class=project-tags-holder><span class="badge btn-info">specialized-community</span></div><div class=project-btn-holder><span><a class="btn btn-outline-info btn-sm" href="https://www.youtube.com/watch?v=MAunk99-EZs&amp;t=9s" target=#>Details</a></span></div></div></div></div></div></div><div class="col-sm-12 col-md-6 col-lg-4 p-2 filtr-item" data-category='all, specialized-community'><div class="card mt-1"><div class=card><a href=https://www.i2sysbio.es/communication/1569/ target=_blank rel=noopener><div class=card-head><img class=card-img-top src=/images/sections/achievements/ecs_hu_6e8df006b573ba08.jpg alt="Early Career Seminars of the I2SysBio."></div><div class=card-header><div><div class=d-flex><h5 class="card-title mb-0">Early Career Seminars of the I2SysBio.</h5></div><div class=sub-title><span>Invited talk</span>
<span>November 2023</span></div></div></div></a><div class="card-body text-justify pt-1 pb-1"><p>Talk about my published work MOSim, a bulk and single-cell multi-omics data simulator.</p><div class=project-card-footer><div class=project-tags-holder><span class="badge btn-info">specialized-community</span></div><div class=project-btn-holder><span><a class="btn btn-outline-info btn-sm" href=https://www.i2sysbio.es/communication/1569/ target=#>Details</a></span></div></div></div></div></div></div><div class="col-sm-12 col-md-6 col-lg-4 p-2 filtr-item" data-category='all, specialized-community'><div class="card mt-1"><div class=card><a href=https://www.rsg-spain.iscbsc.org/ target=_blank rel=noopener><div class=card-head><img class=card-img-top src=/images/sections/outreach/hondigh_hu_f2e1638f59f5e333.jpeg alt="How on earth did I get here, by RSG Spain."></div><div class=card-header><div><div class=d-flex><h5 class="card-title mb-0">How on earth did I get here, by RSG Spain.</h5></div><div class=sub-title><span>Invited talk</span>
<span>April 2023</span></div></div></div></a><div class="card-body text-justify pt-1 pb-1"><p>Talk about my career and give tips and tricks to bioinformatics students of Valencia.</p><div class=project-card-footer><div class=project-tags-holder><span class="badge btn-info">specialized-community</span></div><div class=project-btn-holder><span><a class="btn btn-outline-info btn-sm" href=https://www.rsg-spain.iscbsc.org/ target=#>Details</a></span></div></div></div></div></div></div><div class="col-sm-12 col-md-6 col-lg-4 p-2 filtr-item" data-category='all, general-public'><div class="card mt-1"><div class=card><a href=javascript:void(0)><div class=card-header><div><div class=d-flex><h5 class="card-title mb-0">Radio Show at A Punt station.</h5></div><div class=sub-title><span>Invited talk</span>
<span>April 2021</span></div></div></div></a><div class="card-body text-justify pt-1 pb-1"><p>Talk about the brain drain.</p><div class=project-card-footer><div class=project-tags-holder><span class="badge btn-info">general-public</span></div><div class=project-btn-holder></div></div></div></div></div></div><div class="col-sm-12 col-md-6 col-lg-4 p-2 filtr-item" data-category='all, specialized-community'><div class="card mt-1"><div class=card><a href=javascript:void(0)><div class=card-header><div><div class=d-flex><h5 class="card-title mb-0">LP-101 Lecture Series</h5></div><div class=sub-title><span>Lecturer and workshop organizer</span>
<span>November 2019</span></div></div></div></a><div class="card-body text-justify pt-1 pb-1"><p>Lecture introducing wet-lab aging scientists to the process of going from RNA extraction to count data (bioinformatics).</p><div class=project-card-footer><div class=project-tags-holder><span class="badge btn-info">specialized-community</span></div><div class=project-btn-holder></div></div></div></div></div></div><div class="col-sm-12 col-md-6 col-lg-4 p-2 filtr-item" data-category='all, general-public'><div class="card mt-1"><div class=card><a href="https://genotipia.com/genetica_medica_news/carolina-monzo-entrevista/#:~:text=Carolina%20Monz%C3%B3%3A%20%22%E2%80%9CEl%20mundo,en%20el%20que%20encajar%E2%80%9D%20%2D%20Genotipia" target=_blank rel=noopener><div class=card-header><div><div class=d-flex><h5 class="card-title mb-0">Researcher profile, by Medical Genetics News</h5></div><div class=sub-title><span>Interview</span>
<span>June 2016</span></div></div></div></a><div class="card-body text-justify pt-1 pb-1"><p>Interview about the difficulties of performing science and my latest work on fragile X syndrome.</p><div class=project-card-footer><div class=project-tags-holder><span class="badge btn-info">general-public</span></div><div class=project-btn-holder><span><a class="btn btn-outline-info btn-sm" href="https://genotipia.com/genetica_medica_news/carolina-monzo-entrevista/#:~:text=Carolina%20Monz%C3%B3%3A%20%22%E2%80%9CEl%20mundo,en%20el%20que%20encajar%E2%80%9D%20%2D%20Genotipia" target=#>Details</a></span></div></div></div></div></div></div></div></div></div></div><div class="container-fluid section-holder d-flex bg-secondary"><div class="container-fluid anchor pb-5 recent-posts-section"><h1 class=text-center><span id=featured-posts></span>Other</h1><div class=container><div class=row id=recent-post-cards><div class=post-card><div class=card><div class=card-head><a href=/posts/markdown-sample/ class=post-card-link><img class=card-img-top src=/posts/markdown-sample/hero.svg alt="Hero Image"></a></div><div class=card-body><a href=/posts/markdown-sample/ class=post-card-link><h5 class=card-title>Markdown Samples</h5><p class="card-text post-summary">This is a sample post intended to test the followings:
A different post author. Table of contents. Markdown content rendering. Math rendering. Emoji rendering. Markdown Syntax Rendering Headings The following HTML &lt;h1>—&lt;h6> elements represent six levels of section headings. &lt;h1> is the highest section level while &lt;h6> is the lowest.
H1 H2 H3 H4 H5 H6 Paragraph Xerum, quo qui aut unt expliquam qui dolut labo. Aque venitatiusda cum, voluptionse latur sitiae dolessi aut parist aut dollo enim qui voluptate ma dolestendit peritin re plis aut quas inctum laceat est volestemque commosa as cus endigna tectur, offic to cor sequas etum rerum idem sintibus eiur? Quianimin porecus evelectur, cum que nis nust voloribus ratem aut omnimi, sitatur? Quiatem. Nam, omnis sum am facea corem alique molestrunt et eos evelece arcillit ut aut eos eos nus, sin conecerem erum fuga. Ri oditatquam, ad quibus unda veliamenimin cusam et facea ipsamus es exerum sitate dolores editium rerore eost, temped molorro ratiae volorro te reribus dolorer sperchicium faceata tiustia prat.</p></a><div class=tags><ul style=padding-left:0></ul></div></div><div class=card-footer><span class=float-start>Monday, June 8, 2020
| 3 minutes </span><a href=/posts/markdown-sample/ class="float-end btn btn-outline-info btn-sm">Read</a></div></div></div><div class=post-card><div class=card><div class=card-head><a href=/posts/shortcodes/ class=post-card-link><img class=card-img-top src=/posts/shortcodes/boat.jpg alt="Hero Image"></a></div><div class=card-body><a href=/posts/shortcodes/ class=post-card-link><h5 class=card-title>Shortcodes Samples</h5><p class="card-text post-summary">This is a sample post intended to test the followings:
Default hero image. Different shortcodes. Alert The following alerts are available in this theme.
This is sample alert with type="success". This is sample alert with type="danger". This is sample alert with type="warning". This is sample alert with type="info". This is sample alert with type="dark". This is sample alert with type="primary". This is sample alert with type="secondary". Image A sample image without any attribute. A sample image with height and width attributes. A center aligned image with height and width attributes. A image with float attribute. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras egestas lectus sed leo ultricies ultricies. Praesent tellus risus, eleifend vel efficitur ac, venenatis sit amet sem. Ut ut egestas erat. Fusce ut leo turpis. Morbi consectetur sed lacus vitae vehicula. Cras gravida turpis id eleifend volutpat. Suspendisse nec ipsum eu erat finibus dictum. Morbi volutpat nulla purus, vel maximus ex molestie id. Nullam posuere est urna, at fringilla eros venenatis quis.</p></a><div class=tags><ul style=padding-left:0></ul></div></div><div class=card-footer><span class=float-start>Monday, June 8, 2020
| 3 minutes </span><a href=/posts/shortcodes/ class="float-end btn btn-outline-info btn-sm">Read</a></div></div></div></div></div></div></div><footer id=footer class="container-fluid text-center align-content-center footer pb-2"><div class="container pt-5"><div class="row text-start"><div class="col-md-4 col-sm-12"><h5>Navigation</h5><ul><li class=nav-item><a class=smooth-scroll href=https://carolinamonzo.github.io/#about>About</a></li><li class=nav-item><a class=smooth-scroll href=https://carolinamonzo.github.io/#skills>Skills</a></li><li class=nav-item><a class=smooth-scroll href=https://carolinamonzo.github.io/#experiences>Experience</a></li><li class=nav-item><a class=smooth-scroll href=https://carolinamonzo.github.io/#education>Education</a></li><li class=nav-item><a class=smooth-scroll href=https://carolinamonzo.github.io/#projects>Grants and projects</a></li><li class=nav-item><a class=smooth-scroll href=https://carolinamonzo.github.io/#achievements>Awards</a></li><li class=nav-item><a class=smooth-scroll href=https://carolinamonzo.github.io/#publications>Publications</a></li><li class=nav-item><a class=smooth-scroll href=https://carolinamonzo.github.io/#accomplishments>Courses</a></li><li class=nav-item><a class=smooth-scroll href=https://carolinamonzo.github.io/#outreach>Outreach</a></li><li class=nav-item><a class=smooth-scroll href=https://carolinamonzo.github.io/#featured-posts>Other</a></li></ul></div><div class="col-md-4 col-sm-12"><h5>Contact me:</h5><ul><li><a href=mailto:carolina.monzo.age@gmail.com target=_blank rel=noopener><span><i class="fas fa-envelope"></i></span> <span>carolina.monzo.age@gmail.com</span></a></li><li><a href=https://github.com/carolinamonzo target=_blank rel=noopener><span><i class="fab fa-github"></i></span> <span>carolinamonzo</span></a></li><li><span>Google-Scholar: </span><span>Carolina Monzó</span></li><li><a href=https://www.linkedin.com/in/Carolina%20Monz%c3%b3 target=_blank rel=noopener><span><i class="fab fa-linkedin"></i></span> <span>Carolina Monzó</span></a></li></ul></div></div></div><hr><div class=container><p id=disclaimer><strong>Liability Notice:</strong> This theme is under MIT license. So, you can use it for non-commercial, commercial, or private uses. You can modify or distribute the theme without requiring any permission from the theme author. However, the theme author does not provide any warranty or takes any liability for any issue with the theme.</p></div><hr><div class=container><div class="row text-start"><div class=col-md-4><a id=theme href=https://github.com/hugo-toha/toha target=_blank rel=noopener><img src=/images/theme-logo_hu_b3360284c55cf72d.png alt="Toha Theme Logo">
Toha</a></div><div class="col-md-4 text-center">© 2025 Copyright.</div><div class="col-md-4 text-end"><a id=hugo href=https://gohugo.io/ target=_blank rel=noopener>Powered by
<img src=/images/hugo-logo.svg alt="Hugo Logo" height=18></a></div></div></div></footer><script src=/application.f6959e0d840b2a0cb8ebdc7b64f82c76ed540bafda2e2f31ece1d91e812ba4d1.js integrity="sha256-9pWeDYQLKgy469x7ZPgsdu1UC6/aLi8x7OHZHoErpNE=" defer></script></body></html>